++ ls caseresult/ -ltr
++ wc -l
+ filename=case31
+ mkdir caseresult/case31
+ RESULT_DIR=/home/hadoop/git/physical_design/caseresult/case31
+ TEN=10
+ SIXTY=60
+ SLEEP_TIME=300
+ YCSB_DIR=/home/hadoop/ycsb-0.1.4
+ number_of_operations=5000000
+ number_of_threads=20
+ number_of_records=5000000
+ output_file_suffix=.dat
+ experiment_list_file=/home/hadoop/git/physical_design/case31.list
+ clean=false
+ create=false
+ load=false
+ random_read=false
+ scan=false
+ all=false
+ for i in '$@'
+ '[' all == all ']'
+ all=true
+ let counter=1
+ cd /home/hadoop/ycsb-0.1.4
++ pwd
+ echo /home/hadoop/ycsb-0.1.4
/home/hadoop/ycsb-0.1.4
+ read line
+ true
+ echo 'sleeping for 60 ...'
sleeping for 60 ...
+ sleep 60
+ echo 'droping usertable...'
droping usertable...
+ hbase shell /home/hadoop/git/physical_design/delete_table
SLF4J: Class path contains multiple SLF4J bindings.
SLF4J: Found binding in [jar:file:/home/hadoop/hbase-0.98.3-hadoop1/lib/slf4j-log4j12-1.6.4.jar!/org/slf4j/impl/StaticLoggerBinder.class]
SLF4J: Found binding in [jar:file:/home/hadoop/hadoop-1.2.1/lib/slf4j-log4j12-1.4.3.jar!/org/slf4j/impl/StaticLoggerBinder.class]
SLF4J: See http://www.slf4j.org/codes.html#multiple_bindings for an explanation.
0 row(s) in 4.4850 seconds

0 row(s) in 0.7490 seconds

0 row(s) in 0.0590 seconds

+ echo 'sleeping for 300 ...'
sleeping for 300 ...
+ sleep 300
+ true
+ echo 'creating usertable...'
creating usertable...
+ create_statement='create '\''usertable'\'', { NAME=>'\''family'\'', CONFIGURATION =>  { '\''hbase.hstore.compaction.min'\'' => '\''9'\'', '\''hbase.hregion.majorcompaction'\'' => '\''0'\'' } }, SPLITS => ['\''user1'\'', '\''user2'\'', '\''user3'\'', '\''user4'\'', '\''user5'\'', '\''user6'\'', '\''user7'\'', '\''user8'\'', '\''user9'\'']'
+ echo create ''\''usertable'\'',' '{' 'NAME=>'\''family'\'',' CONFIGURATION '=>' '{' ''\''hbase.hstore.compaction.min'\''' '=>' ''\''9'\'',' ''\''hbase.hregion.majorcompaction'\''' '=>' ''\''0'\''' '}' '},' SPLITS '=>' '['\''user1'\'',' ''\''user2'\'',' ''\''user3'\'',' ''\''user4'\'',' ''\''user5'\'',' ''\''user6'\'',' ''\''user7'\'',' ''\''user8'\'',' ''\''user9'\'']'
+ echo 'exit '
+ create_table /home/hadoop/git/physical_design/caseresult/case31/create_1.script
+ echo 'creating table...'
creating table...
+ echo /home/hadoop/git/physical_design/caseresult/case31/create_1.script
/home/hadoop/git/physical_design/caseresult/case31/create_1.script
+ hbase shell /home/hadoop/git/physical_design/caseresult/case31/create_1.script
SLF4J: Class path contains multiple SLF4J bindings.
SLF4J: Found binding in [jar:file:/home/hadoop/hbase-0.98.3-hadoop1/lib/slf4j-log4j12-1.6.4.jar!/org/slf4j/impl/StaticLoggerBinder.class]
SLF4J: Found binding in [jar:file:/home/hadoop/hadoop-1.2.1/lib/slf4j-log4j12-1.4.3.jar!/org/slf4j/impl/StaticLoggerBinder.class]
SLF4J: See http://www.slf4j.org/codes.html#multiple_bindings for an explanation.
0 row(s) in 1.8140 seconds

+ echo 'sleeping for 10 sec ...'
sleeping for 10 sec ...
+ sleep 10
+ true
+ echo 'loading data...'
loading data...
+ echo 'starting time: '
starting time: 
++ date +%s
+ echo 1404810175
1404810175
++ echo /home/hadoop/git/physical_design/caseresult/case31/load_1.dat
+ ./ld 5000000 20 /home/hadoop/git/physical_design/caseresult/case31/load_1.dat
Loading workload...
Starting test.
2014-07-08 02:02:55:864 0 sec: 0 operations; 
14/07/08 02:02:56 INFO zookeeper.ZooKeeper: Client environment:zookeeper.version=3.4.6-1569965, built on 02/20/2014 09:09 GMT
14/07/08 02:02:56 INFO zookeeper.ZooKeeper: Client environment:host.name=sceplus-vm48.almaden.ibm.com
14/07/08 02:02:56 INFO zookeeper.ZooKeeper: Client environment:java.version=1.6.0_31
14/07/08 02:02:56 INFO zookeeper.ZooKeeper: Client environment:java.vendor=Sun Microsystems Inc.
14/07/08 02:02:56 INFO zookeeper.ZooKeeper: Client environment:java.home=/usr/lib/jvm/java-6-openjdk-amd64/jre
14/07/08 02:02:56 INFO zookeeper.ZooKeeper: Client environment:java.class.path=/home/hadoop/ycsb-0.1.4/hbase-binding/lib/hbase-binding-0.1.4.jar:/home/hadoop/ycsb-0.1.4/hbase-binding/conf:/home/hadoop/ycsb-0.1.4/jdbc-binding/conf:/home/hadoop/ycsb-0.1.4/core/lib/core-0.1.4.jar
14/07/08 02:02:56 INFO zookeeper.ZooKeeper: Client environment:java.library.path=/usr/lib/jvm/java-6-openjdk-amd64/jre/lib/amd64/server:/usr/lib/jvm/java-6-openjdk-amd64/jre/lib/amd64:/usr/lib/jvm/java-6-openjdk-amd64/jre/../lib/amd64:/usr/java/packages/lib/amd64:/usr/lib/jni:/lib:/usr/lib
14/07/08 02:02:56 INFO zookeeper.ZooKeeper: Client environment:java.io.tmpdir=/tmp
14/07/08 02:02:56 INFO zookeeper.ZooKeeper: Client environment:java.compiler=<NA>
14/07/08 02:02:56 INFO zookeeper.ZooKeeper: Client environment:os.name=Linux
14/07/08 02:02:56 INFO zookeeper.ZooKeeper: Client environment:os.arch=amd64
14/07/08 02:02:56 INFO zookeeper.ZooKeeper: Client environment:os.version=3.13.0-24-generic
14/07/08 02:02:56 INFO zookeeper.ZooKeeper: Client environment:user.name=hadoop
14/07/08 02:02:56 INFO zookeeper.ZooKeeper: Client environment:user.home=/home/hadoop
14/07/08 02:02:56 INFO zookeeper.ZooKeeper: Client environment:user.dir=/home/hadoop/ycsb-0.1.4
14/07/08 02:02:56 INFO zookeeper.ZooKeeper: Initiating client connection, connectString=sceplus-vm49.almaden.ibm.com:2181,sceplus-vm48.almaden.ibm.com:2181 sessionTimeout=90000 watcher=hconnection-0xc2c915f, quorum=sceplus-vm49.almaden.ibm.com:2181,sceplus-vm48.almaden.ibm.com:2181, baseZNode=/hbase
14/07/08 02:02:56 INFO zookeeper.RecoverableZooKeeper: Process identifier=hconnection-0xc2c915f connecting to ZooKeeper ensemble=sceplus-vm49.almaden.ibm.com:2181,sceplus-vm48.almaden.ibm.com:2181
14/07/08 02:02:56 INFO zookeeper.ClientCnxn: Opening socket connection to server slave1/9.1.143.59:2181. Will not attempt to authenticate using SASL (unknown error)
14/07/08 02:02:56 INFO zookeeper.ClientCnxn: Socket connection established to slave1/9.1.143.59:2181, initiating session
14/07/08 02:02:56 INFO zookeeper.ClientCnxn: Session establishment complete on server slave1/9.1.143.59:2181, sessionid = 0x1471530370b000c, negotiated timeout = 90000
2014-07-08 02:03:05:866 10 sec: 257253 operations; 25720.16 current ops/sec; [INSERT AverageLatency(us)=337.08] 
2014-07-08 02:03:15:867 20 sec: 258260 operations; 100.69 current ops/sec; [INSERT AverageLatency(us)=15.21] 
2014-07-08 02:03:25:868 30 sec: 258260 operations; 0 current ops/sec;  
2014-07-08 02:03:35:869 40 sec: 258260 operations; 0 current ops/sec;  
2014-07-08 02:03:45:869 50 sec: 258260 operations; 0 current ops/sec;  
14/07/08 02:03:55 INFO client.AsyncProcess: #15, table=usertable, attempt=10/35 failed 2469 ops, last exception: java.net.SocketTimeoutException: Call to sceplus-vm48.almaden.ibm.com/9.1.143.58:60020 failed because java.net.SocketTimeoutException: 2000 millis timeout while waiting for channel to be ready for read. ch : java.nio.channels.SocketChannel[connected local=/9.1.143.58:54458 remote=sceplus-vm48.almaden.ibm.com/9.1.143.58:60020] on sceplus-vm48.almaden.ibm.com,60020,1404809787584, tracking started Tue Jul 08 02:03:02 PDT 2014, retrying after 10058 ms, replay 2469 ops.
14/07/08 02:03:55 INFO client.AsyncProcess: #1, table=usertable, attempt=10/35 failed 2371 ops, last exception: java.net.SocketTimeoutException: Call to sceplus-vm48.almaden.ibm.com/9.1.143.58:60020 failed because java.net.SocketTimeoutException: 2000 millis timeout while waiting for channel to be ready for read. ch : java.nio.channels.SocketChannel[connected local=/9.1.143.58:54458 remote=sceplus-vm48.almaden.ibm.com/9.1.143.58:60020] on sceplus-vm48.almaden.ibm.com,60020,1404809787584, tracking started Tue Jul 08 02:02:58 PDT 2014, retrying after 10011 ms, replay 2371 ops.
2014-07-08 02:03:55:870 60 sec: 258260 operations; 0 current ops/sec;  
14/07/08 02:03:57 INFO client.AsyncProcess: #19, table=usertable, attempt=10/35 failed 2449 ops, last exception: java.io.IOException: Call to sceplus-vm48.almaden.ibm.com/9.1.143.58:60020 failed on local exception: org.apache.hadoop.hbase.ipc.RpcClient$CallTimeoutException: Call id=401, waitTime=3496, rpcTimetout=2000 on sceplus-vm48.almaden.ibm.com,60020,1404809787584, tracking started Tue Jul 08 02:03:02 PDT 2014, retrying after 10031 ms, replay 2449 ops.
14/07/08 02:03:57 INFO client.AsyncProcess: #11, table=usertable, attempt=10/35 failed 2406 ops, last exception: java.io.IOException: Call to sceplus-vm48.almaden.ibm.com/9.1.143.58:60020 failed on local exception: org.apache.hadoop.hbase.ipc.RpcClient$CallTimeoutException: Call id=401, waitTime=3496, rpcTimetout=2000 on sceplus-vm48.almaden.ibm.com,60020,1404809787584, tracking started Tue Jul 08 02:02:58 PDT 2014, retrying after 10049 ms, replay 2406 ops.
14/07/08 02:03:57 INFO client.AsyncProcess: #16, table=usertable, attempt=10/35 failed 2443 ops, last exception: java.io.IOException: Call to sceplus-vm48.almaden.ibm.com/9.1.143.58:60020 failed on local exception: org.apache.hadoop.hbase.ipc.RpcClient$CallTimeoutException: Call id=401, waitTime=3496, rpcTimetout=2000 on sceplus-vm48.almaden.ibm.com,60020,1404809787584, tracking started Tue Jul 08 02:02:58 PDT 2014, retrying after 10056 ms, replay 2443 ops.
14/07/08 02:03:57 INFO client.AsyncProcess: #14, table=usertable, attempt=10/35 failed 2412 ops, last exception: java.io.IOException: Call to sceplus-vm48.almaden.ibm.com/9.1.143.58:60020 failed on local exception: org.apache.hadoop.hbase.ipc.RpcClient$CallTimeoutException: Call id=401, waitTime=3496, rpcTimetout=2000 on sceplus-vm48.almaden.ibm.com,60020,1404809787584, tracking started Tue Jul 08 02:02:58 PDT 2014, retrying after 10004 ms, replay 2412 ops.
14/07/08 02:03:57 INFO client.AsyncProcess: #10, table=usertable, attempt=10/35 failed 2427 ops, last exception: java.io.IOException: Call to sceplus-vm48.almaden.ibm.com/9.1.143.58:60020 failed on local exception: org.apache.hadoop.hbase.ipc.RpcClient$CallTimeoutException: Call id=401, waitTime=3496, rpcTimetout=2000 on sceplus-vm48.almaden.ibm.com,60020,1404809787584, tracking started Tue Jul 08 02:02:58 PDT 2014, retrying after 10004 ms, replay 2427 ops.
14/07/08 02:03:57 INFO client.AsyncProcess: #18, table=usertable, attempt=10/35 failed 2426 ops, last exception: java.io.IOException: Call to sceplus-vm48.almaden.ibm.com/9.1.143.58:60020 failed on local exception: org.apache.hadoop.hbase.ipc.RpcClient$CallTimeoutException: Call id=401, waitTime=3496, rpcTimetout=2000 on sceplus-vm48.almaden.ibm.com,60020,1404809787584, tracking started Tue Jul 08 02:02:58 PDT 2014, retrying after 10029 ms, replay 2426 ops.
14/07/08 02:03:57 INFO client.AsyncProcess: #6, table=usertable, attempt=10/35 failed 2463 ops, last exception: java.io.IOException: Call to sceplus-vm48.almaden.ibm.com/9.1.143.58:60020 failed on local exception: org.apache.hadoop.hbase.ipc.RpcClient$CallTimeoutException: Call id=401, waitTime=3496, rpcTimetout=2000 on sceplus-vm48.almaden.ibm.com,60020,1404809787584, tracking started Tue Jul 08 02:03:02 PDT 2014, retrying after 10056 ms, replay 2463 ops.
14/07/08 02:03:57 INFO client.AsyncProcess: #3, table=usertable, attempt=10/35 failed 2469 ops, last exception: java.io.IOException: Call to sceplus-vm48.almaden.ibm.com/9.1.143.58:60020 failed on local exception: org.apache.hadoop.hbase.ipc.RpcClient$CallTimeoutException: Call id=401, waitTime=3496, rpcTimetout=2000 on sceplus-vm48.almaden.ibm.com,60020,1404809787584, tracking started Tue Jul 08 02:02:58 PDT 2014, retrying after 10042 ms, replay 2469 ops.
14/07/08 02:03:57 INFO client.AsyncProcess: #7, table=usertable, attempt=10/35 failed 2456 ops, last exception: java.io.IOException: Call to sceplus-vm48.almaden.ibm.com/9.1.143.58:60020 failed on local exception: org.apache.hadoop.hbase.ipc.RpcClient$CallTimeoutException: Call id=401, waitTime=3496, rpcTimetout=2000 on sceplus-vm48.almaden.ibm.com,60020,1404809787584, tracking started Tue Jul 08 02:03:02 PDT 2014, retrying after 10042 ms, replay 2456 ops.
14/07/08 02:03:57 INFO client.AsyncProcess: #6, table=usertable, attempt=10/35 failed 2404 ops, last exception: java.io.IOException: Call to sceplus-vm48.almaden.ibm.com/9.1.143.58:60020 failed on local exception: org.apache.hadoop.hbase.ipc.RpcClient$CallTimeoutException: Call id=401, waitTime=3496, rpcTimetout=2000 on sceplus-vm48.almaden.ibm.com,60020,1404809787584, tracking started Tue Jul 08 02:02:58 PDT 2014, retrying after 10092 ms, replay 2404 ops.
14/07/08 02:03:57 INFO client.AsyncProcess: #15, table=usertable, attempt=10/35 failed 2420 ops, last exception: java.io.IOException: Call to sceplus-vm48.almaden.ibm.com/9.1.143.58:60020 failed on local exception: org.apache.hadoop.hbase.ipc.RpcClient$CallTimeoutException: Call id=401, waitTime=3496, rpcTimetout=2000 on sceplus-vm48.almaden.ibm.com,60020,1404809787584, tracking started Tue Jul 08 02:02:58 PDT 2014, retrying after 10041 ms, replay 2420 ops.
14/07/08 02:03:57 INFO client.AsyncProcess: #2, table=usertable, attempt=10/35 failed 2452 ops, last exception: java.io.IOException: Call to sceplus-vm48.almaden.ibm.com/9.1.143.58:60020 failed on local exception: org.apache.hadoop.hbase.ipc.RpcClient$CallTimeoutException: Call id=401, waitTime=3496, rpcTimetout=2000 on sceplus-vm48.almaden.ibm.com,60020,1404809787584, tracking started Tue Jul 08 02:02:58 PDT 2014, retrying after 10057 ms, replay 2452 ops.
14/07/08 02:03:57 INFO client.AsyncProcess: #9, table=usertable, attempt=10/35 failed 2422 ops, last exception: java.io.IOException: Call to sceplus-vm48.almaden.ibm.com/9.1.143.58:60020 failed on local exception: org.apache.hadoop.hbase.ipc.RpcClient$CallTimeoutException: Call id=401, waitTime=3496, rpcTimetout=2000 on sceplus-vm48.almaden.ibm.com,60020,1404809787584, tracking started Tue Jul 08 02:03:02 PDT 2014, retrying after 10022 ms, replay 2422 ops.
14/07/08 02:03:57 INFO client.AsyncProcess: #2, table=usertable, attempt=10/35 failed 2435 ops, last exception: java.io.IOException: Call to sceplus-vm48.almaden.ibm.com/9.1.143.58:60020 failed on local exception: org.apache.hadoop.hbase.ipc.RpcClient$CallTimeoutException: Call id=401, waitTime=3496, rpcTimetout=2000 on sceplus-vm48.almaden.ibm.com,60020,1404809787584, tracking started Tue Jul 08 02:03:02 PDT 2014, retrying after 10085 ms, replay 2435 ops.
14/07/08 02:03:57 INFO client.AsyncProcess: #8, table=usertable, attempt=10/35 failed 2430 ops, last exception: java.io.IOException: Call to sceplus-vm48.almaden.ibm.com/9.1.143.58:60020 failed on local exception: org.apache.hadoop.hbase.ipc.RpcClient$CallTimeoutException: Call id=401, waitTime=3496, rpcTimetout=2000 on sceplus-vm48.almaden.ibm.com,60020,1404809787584, tracking started Tue Jul 08 02:02:58 PDT 2014, retrying after 10008 ms, replay 2430 ops.
14/07/08 02:03:57 INFO client.AsyncProcess: #4, table=usertable, attempt=10/35 failed 2478 ops, last exception: java.io.IOException: Call to sceplus-vm48.almaden.ibm.com/9.1.143.58:60020 failed on local exception: org.apache.hadoop.hbase.ipc.RpcClient$CallTimeoutException: Call id=401, waitTime=3496, rpcTimetout=2000 on sceplus-vm48.almaden.ibm.com,60020,1404809787584, tracking started Tue Jul 08 02:02:58 PDT 2014, retrying after 10009 ms, replay 2478 ops.
14/07/08 02:03:57 INFO client.AsyncProcess: #14, table=usertable, attempt=10/35 failed 2396 ops, last exception: java.io.IOException: Call to sceplus-vm48.almaden.ibm.com/9.1.143.58:60020 failed on local exception: org.apache.hadoop.hbase.ipc.RpcClient$CallTimeoutException: Call id=401, waitTime=3496, rpcTimetout=2000 on sceplus-vm48.almaden.ibm.com,60020,1404809787584, tracking started Tue Jul 08 02:03:02 PDT 2014, retrying after 10033 ms, replay 2396 ops.
14/07/08 02:03:57 INFO client.AsyncProcess: #12, table=usertable, attempt=10/35 failed 2445 ops, last exception: java.io.IOException: Call to sceplus-vm48.almaden.ibm.com/9.1.143.58:60020 failed on local exception: org.apache.hadoop.hbase.ipc.RpcClient$CallTimeoutException: Call id=401, waitTime=3496, rpcTimetout=2000 on sceplus-vm48.almaden.ibm.com,60020,1404809787584, tracking started Tue Jul 08 02:03:02 PDT 2014, retrying after 10065 ms, replay 2445 ops.
14/07/08 02:03:57 INFO client.AsyncProcess: #17, table=usertable, attempt=10/35 failed 2411 ops, last exception: java.io.IOException: Call to sceplus-vm48.almaden.ibm.com/9.1.143.58:60020 failed on local exception: org.apache.hadoop.hbase.ipc.RpcClient$CallTimeoutException: Call id=401, waitTime=3496, rpcTimetout=2000 on sceplus-vm48.almaden.ibm.com,60020,1404809787584, tracking started Tue Jul 08 02:02:58 PDT 2014, retrying after 10049 ms, replay 2411 ops.
14/07/08 02:03:57 INFO client.AsyncProcess: #20, table=usertable, attempt=10/35 failed 2445 ops, last exception: java.io.IOException: Call to sceplus-vm48.almaden.ibm.com/9.1.143.58:60020 failed on local exception: org.apache.hadoop.hbase.ipc.RpcClient$CallTimeoutException: Call id=401, waitTime=3496, rpcTimetout=2000 on sceplus-vm48.almaden.ibm.com,60020,1404809787584, tracking started Tue Jul 08 02:02:58 PDT 2014, retrying after 10044 ms, replay 2445 ops.
14/07/08 02:03:57 INFO client.AsyncProcess: #12, table=usertable, attempt=10/35 failed 2431 ops, last exception: java.io.IOException: Call to sceplus-vm48.almaden.ibm.com/9.1.143.58:60020 failed on local exception: org.apache.hadoop.hbase.ipc.RpcClient$CallTimeoutException: Call id=401, waitTime=3496, rpcTimetout=2000 on sceplus-vm48.almaden.ibm.com,60020,1404809787584, tracking started Tue Jul 08 02:02:58 PDT 2014, retrying after 10089 ms, replay 2431 ops.
14/07/08 02:03:57 INFO client.AsyncProcess: #13, table=usertable, attempt=10/35 failed 2425 ops, last exception: java.io.IOException: Call to sceplus-vm48.almaden.ibm.com/9.1.143.58:60020 failed on local exception: org.apache.hadoop.hbase.ipc.RpcClient$CallTimeoutException: Call id=401, waitTime=3496, rpcTimetout=2000 on sceplus-vm48.almaden.ibm.com,60020,1404809787584, tracking started Tue Jul 08 02:02:58 PDT 2014, retrying after 10094 ms, replay 2425 ops.
14/07/08 02:03:58 INFO client.AsyncProcess: #5, table=usertable, attempt=10/35 failed 2411 ops, last exception: java.io.IOException: Call to sceplus-vm48.almaden.ibm.com/9.1.143.58:60020 failed on local exception: org.apache.hadoop.hbase.ipc.RpcClient$CallTimeoutException: Call id=423, waitTime=2612, rpcTimetout=2000 on sceplus-vm48.almaden.ibm.com,60020,1404809787584, tracking started Tue Jul 08 02:03:02 PDT 2014, retrying after 10087 ms, replay 2411 ops.
14/07/08 02:03:58 INFO client.AsyncProcess: #9, table=usertable, attempt=10/35 failed 2466 ops, last exception: java.io.IOException: Call to sceplus-vm48.almaden.ibm.com/9.1.143.58:60020 failed on local exception: org.apache.hadoop.hbase.ipc.RpcClient$CallTimeoutException: Call id=423, waitTime=2612, rpcTimetout=2000 on sceplus-vm48.almaden.ibm.com,60020,1404809787584, tracking started Tue Jul 08 02:03:03 PDT 2014, retrying after 10039 ms, replay 2466 ops.
14/07/08 02:03:58 INFO client.AsyncProcess: #7, table=usertable, attempt=10/35 failed 2458 ops, last exception: java.io.IOException: Call to sceplus-vm48.almaden.ibm.com/9.1.143.58:60020 failed on local exception: org.apache.hadoop.hbase.ipc.RpcClient$CallTimeoutException: Call id=423, waitTime=2612, rpcTimetout=2000 on sceplus-vm48.almaden.ibm.com,60020,1404809787584, tracking started Tue Jul 08 02:03:05 PDT 2014, retrying after 10057 ms, replay 2458 ops.
14/07/08 02:03:58 INFO client.AsyncProcess: #5, table=usertable, attempt=10/35 failed 2410 ops, last exception: java.io.IOException: Call to sceplus-vm48.almaden.ibm.com/9.1.143.58:60020 failed on local exception: org.apache.hadoop.hbase.ipc.RpcClient$CallTimeoutException: Call id=423, waitTime=2612, rpcTimetout=2000 on sceplus-vm48.almaden.ibm.com,60020,1404809787584, tracking started Tue Jul 08 02:03:05 PDT 2014, retrying after 10096 ms, replay 2410 ops.
14/07/08 02:03:58 INFO client.AsyncProcess: #12, table=usertable, attempt=10/35 failed 2413 ops, last exception: java.io.IOException: Call to sceplus-vm48.almaden.ibm.com/9.1.143.58:60020 failed on local exception: org.apache.hadoop.hbase.ipc.RpcClient$CallTimeoutException: Call id=423, waitTime=2612, rpcTimetout=2000 on sceplus-vm48.almaden.ibm.com,60020,1404809787584, tracking started Tue Jul 08 02:03:05 PDT 2014, retrying after 10090 ms, replay 2413 ops.
2014-07-08 02:04:05:871 70 sec: 258260 operations; 0 current ops/sec;  
14/07/08 02:04:06 INFO client.AsyncProcess: #15, table=usertable, attempt=11/35 SUCCEEDED on sceplus-vm48.almaden.ibm.com,60020,1404809787584, tracking started Tue Jul 08 02:03:02 PDT 2014
14/07/08 02:04:06 INFO client.AsyncProcess: #1, table=usertable, attempt=11/35 SUCCEEDED on sceplus-vm48.almaden.ibm.com,60020,1404809787584, tracking started Tue Jul 08 02:02:58 PDT 2014
14/07/08 02:04:09 INFO client.AsyncProcess: #19, table=usertable, attempt=11/35 failed 2449 ops, last exception: java.io.IOException: Call to sceplus-vm48.almaden.ibm.com/9.1.143.58:60020 failed on local exception: org.apache.hadoop.hbase.ipc.RpcClient$CallTimeoutException: Call id=438, waitTime=2419, rpcTimetout=2000 on sceplus-vm48.almaden.ibm.com,60020,1404809787584, tracking started Tue Jul 08 02:03:02 PDT 2014, retrying after 10016 ms, replay 2449 ops.
14/07/08 02:04:09 INFO client.AsyncProcess: #12, table=usertable, attempt=11/35 failed 2445 ops, last exception: java.io.IOException: Call to sceplus-vm48.almaden.ibm.com/9.1.143.58:60020 failed on local exception: org.apache.hadoop.hbase.ipc.RpcClient$CallTimeoutException: Call id=438, waitTime=2419, rpcTimetout=2000 on sceplus-vm48.almaden.ibm.com,60020,1404809787584, tracking started Tue Jul 08 02:03:02 PDT 2014, retrying after 10031 ms, replay 2445 ops.
14/07/08 02:04:09 INFO client.AsyncProcess: #16, table=usertable, attempt=11/35 failed 2443 ops, last exception: java.io.IOException: Call to sceplus-vm48.almaden.ibm.com/9.1.143.58:60020 failed on local exception: org.apache.hadoop.hbase.ipc.RpcClient$CallTimeoutException: Call id=438, waitTime=2419, rpcTimetout=2000 on sceplus-vm48.almaden.ibm.com,60020,1404809787584, tracking started Tue Jul 08 02:02:58 PDT 2014, retrying after 10081 ms, replay 2443 ops.
14/07/08 02:04:09 INFO client.AsyncProcess: #14, table=usertable, attempt=11/35 failed 2412 ops, last exception: java.io.IOException: Call to sceplus-vm48.almaden.ibm.com/9.1.143.58:60020 failed on local exception: org.apache.hadoop.hbase.ipc.RpcClient$CallTimeoutException: Call id=438, waitTime=2419, rpcTimetout=2000 on sceplus-vm48.almaden.ibm.com,60020,1404809787584, tracking started Tue Jul 08 02:02:58 PDT 2014, retrying after 10073 ms, replay 2412 ops.
14/07/08 02:04:09 INFO client.AsyncProcess: #10, table=usertable, attempt=11/35 failed 2427 ops, last exception: java.io.IOException: Call to sceplus-vm48.almaden.ibm.com/9.1.143.58:60020 failed on local exception: org.apache.hadoop.hbase.ipc.RpcClient$CallTimeoutException: Call id=438, waitTime=2419, rpcTimetout=2000 on sceplus-vm48.almaden.ibm.com,60020,1404809787584, tracking started Tue Jul 08 02:02:58 PDT 2014, retrying after 10060 ms, replay 2427 ops.
14/07/08 02:04:09 INFO client.AsyncProcess: #9, table=usertable, attempt=11/35 SUCCEEDED on sceplus-vm48.almaden.ibm.com,60020,1404809787584, tracking started Tue Jul 08 02:03:02 PDT 2014
14/07/08 02:04:09 INFO client.AsyncProcess: #18, table=usertable, attempt=11/35 failed 2426 ops, last exception: java.io.IOException: Call to sceplus-vm48.almaden.ibm.com/9.1.143.58:60020 failed on local exception: org.apache.hadoop.hbase.ipc.RpcClient$CallTimeoutException: Call id=438, waitTime=2419, rpcTimetout=2000 on sceplus-vm48.almaden.ibm.com,60020,1404809787584, tracking started Tue Jul 08 02:02:58 PDT 2014, retrying after 10085 ms, replay 2426 ops.
14/07/08 02:04:09 INFO client.AsyncProcess: #20, table=usertable, attempt=11/35 failed 2445 ops, last exception: java.io.IOException: Call to sceplus-vm48.almaden.ibm.com/9.1.143.58:60020 failed on local exception: org.apache.hadoop.hbase.ipc.RpcClient$CallTimeoutException: Call id=438, waitTime=2419, rpcTimetout=2000 on sceplus-vm48.almaden.ibm.com,60020,1404809787584, tracking started Tue Jul 08 02:02:58 PDT 2014, retrying after 10051 ms, replay 2445 ops.
14/07/08 02:04:09 INFO client.AsyncProcess: #4, table=usertable, attempt=11/35 failed 2478 ops, last exception: java.io.IOException: Call to sceplus-vm48.almaden.ibm.com/9.1.143.58:60020 failed on local exception: org.apache.hadoop.hbase.ipc.RpcClient$CallTimeoutException: Call id=438, waitTime=2419, rpcTimetout=2000 on sceplus-vm48.almaden.ibm.com,60020,1404809787584, tracking started Tue Jul 08 02:02:58 PDT 2014, retrying after 10023 ms, replay 2478 ops.
14/07/08 02:04:09 INFO client.AsyncProcess: #2, table=usertable, attempt=11/35 failed 2452 ops, last exception: java.io.IOException: Call to sceplus-vm48.almaden.ibm.com/9.1.143.58:60020 failed on local exception: org.apache.hadoop.hbase.ipc.RpcClient$CallTimeoutException: Call id=438, waitTime=2419, rpcTimetout=2000 on sceplus-vm48.almaden.ibm.com,60020,1404809787584, tracking started Tue Jul 08 02:02:58 PDT 2014, retrying after 10015 ms, replay 2452 ops.
14/07/08 02:04:09 INFO client.AsyncProcess: #14, table=usertable, attempt=11/35 failed 2396 ops, last exception: java.io.IOException: Call to sceplus-vm48.almaden.ibm.com/9.1.143.58:60020 failed on local exception: org.apache.hadoop.hbase.ipc.RpcClient$CallTimeoutException: Call id=438, waitTime=2419, rpcTimetout=2000 on sceplus-vm48.almaden.ibm.com,60020,1404809787584, tracking started Tue Jul 08 02:03:02 PDT 2014, retrying after 10011 ms, replay 2396 ops.
14/07/08 02:04:09 INFO client.AsyncProcess: #8, table=usertable, attempt=11/35 failed 2430 ops, last exception: java.io.IOException: Call to sceplus-vm48.almaden.ibm.com/9.1.143.58:60020 failed on local exception: org.apache.hadoop.hbase.ipc.RpcClient$CallTimeoutException: Call id=438, waitTime=2419, rpcTimetout=2000 on sceplus-vm48.almaden.ibm.com,60020,1404809787584, tracking started Tue Jul 08 02:02:58 PDT 2014, retrying after 10054 ms, replay 2430 ops.
14/07/08 02:04:09 INFO client.AsyncProcess: #3, table=usertable, attempt=11/35 failed 2469 ops, last exception: java.io.IOException: Call to sceplus-vm48.almaden.ibm.com/9.1.143.58:60020 failed on local exception: org.apache.hadoop.hbase.ipc.RpcClient$CallTimeoutException: Call id=438, waitTime=2419, rpcTimetout=2000 on sceplus-vm48.almaden.ibm.com,60020,1404809787584, tracking started Tue Jul 08 02:02:58 PDT 2014, retrying after 10069 ms, replay 2469 ops.
14/07/08 02:04:09 INFO client.AsyncProcess: #15, table=usertable, attempt=11/35 failed 2420 ops, last exception: java.io.IOException: Call to sceplus-vm48.almaden.ibm.com/9.1.143.58:60020 failed on local exception: org.apache.hadoop.hbase.ipc.RpcClient$CallTimeoutException: Call id=438, waitTime=2419, rpcTimetout=2000 on sceplus-vm48.almaden.ibm.com,60020,1404809787584, tracking started Tue Jul 08 02:02:58 PDT 2014, retrying after 10042 ms, replay 2420 ops.
14/07/08 02:04:09 INFO client.AsyncProcess: #13, table=usertable, attempt=11/35 failed 2425 ops, last exception: java.io.IOException: Call to sceplus-vm48.almaden.ibm.com/9.1.143.58:60020 failed on local exception: org.apache.hadoop.hbase.ipc.RpcClient$CallTimeoutException: Call id=438, waitTime=2419, rpcTimetout=2000 on sceplus-vm48.almaden.ibm.com,60020,1404809787584, tracking started Tue Jul 08 02:02:58 PDT 2014, retrying after 10054 ms, replay 2425 ops.
14/07/08 02:04:09 INFO client.AsyncProcess: #11, table=usertable, attempt=11/35 failed 2406 ops, last exception: java.io.IOException: Call to sceplus-vm48.almaden.ibm.com/9.1.143.58:60020 failed on local exception: org.apache.hadoop.hbase.ipc.RpcClient$CallTimeoutException: Call id=438, waitTime=2419, rpcTimetout=2000 on sceplus-vm48.almaden.ibm.com,60020,1404809787584, tracking started Tue Jul 08 02:02:58 PDT 2014, retrying after 10090 ms, replay 2406 ops.
14/07/08 02:04:09 INFO client.AsyncProcess: #7, table=usertable, attempt=11/35 failed 2456 ops, last exception: java.io.IOException: Call to sceplus-vm48.almaden.ibm.com/9.1.143.58:60020 failed on local exception: org.apache.hadoop.hbase.ipc.RpcClient$CallTimeoutException: Call id=438, waitTime=2419, rpcTimetout=2000 on sceplus-vm48.almaden.ibm.com,60020,1404809787584, tracking started Tue Jul 08 02:03:02 PDT 2014, retrying after 10095 ms, replay 2456 ops.
14/07/08 02:04:09 INFO client.AsyncProcess: #6, table=usertable, attempt=11/35 failed 2404 ops, last exception: java.io.IOException: Call to sceplus-vm48.almaden.ibm.com/9.1.143.58:60020 failed on local exception: org.apache.hadoop.hbase.ipc.RpcClient$CallTimeoutException: Call id=438, waitTime=2419, rpcTimetout=2000 on sceplus-vm48.almaden.ibm.com,60020,1404809787584, tracking started Tue Jul 08 02:02:58 PDT 2014, retrying after 10049 ms, replay 2404 ops.
14/07/08 02:04:09 INFO client.AsyncProcess: #12, table=usertable, attempt=11/35 failed 2431 ops, last exception: java.io.IOException: Call to sceplus-vm48.almaden.ibm.com/9.1.143.58:60020 failed on local exception: org.apache.hadoop.hbase.ipc.RpcClient$CallTimeoutException: Call id=438, waitTime=2419, rpcTimetout=2000 on sceplus-vm48.almaden.ibm.com,60020,1404809787584, tracking started Tue Jul 08 02:02:58 PDT 2014, retrying after 10079 ms, replay 2431 ops.
14/07/08 02:04:09 INFO client.AsyncProcess: #2, table=usertable, attempt=11/35 failed 2435 ops, last exception: java.io.IOException: Call to sceplus-vm48.almaden.ibm.com/9.1.143.58:60020 failed on local exception: org.apache.hadoop.hbase.ipc.RpcClient$CallTimeoutException: Call id=438, waitTime=2419, rpcTimetout=2000 on sceplus-vm48.almaden.ibm.com,60020,1404809787584, tracking started Tue Jul 08 02:03:02 PDT 2014, retrying after 10080 ms, replay 2435 ops.
14/07/08 02:04:09 INFO client.AsyncProcess: #6, table=usertable, attempt=11/35 failed 2463 ops, last exception: java.io.IOException: Call to sceplus-vm48.almaden.ibm.com/9.1.143.58:60020 failed on local exception: org.apache.hadoop.hbase.ipc.RpcClient$CallTimeoutException: Call id=438, waitTime=2419, rpcTimetout=2000 on sceplus-vm48.almaden.ibm.com,60020,1404809787584, tracking started Tue Jul 08 02:03:02 PDT 2014, retrying after 10058 ms, replay 2463 ops.
14/07/08 02:04:09 INFO client.AsyncProcess: #17, table=usertable, attempt=11/35 failed 2411 ops, last exception: java.io.IOException: Call to sceplus-vm48.almaden.ibm.com/9.1.143.58:60020 failed on local exception: org.apache.hadoop.hbase.ipc.RpcClient$CallTimeoutException: Call id=438, waitTime=2419, rpcTimetout=2000 on sceplus-vm48.almaden.ibm.com,60020,1404809787584, tracking started Tue Jul 08 02:02:58 PDT 2014, retrying after 10053 ms, replay 2411 ops.
14/07/08 02:04:10 INFO client.AsyncProcess: #9, table=usertable, attempt=11/35 failed 2466 ops, last exception: java.io.IOException: Call to sceplus-vm48.almaden.ibm.com/9.1.143.58:60020 failed on local exception: org.apache.hadoop.hbase.ipc.RpcClient$CallTimeoutException: Call id=460, waitTime=2007, rpcTimetout=2000 on sceplus-vm48.almaden.ibm.com,60020,1404809787584, tracking started Tue Jul 08 02:03:03 PDT 2014, retrying after 10020 ms, replay 2466 ops.
14/07/08 02:04:10 INFO client.AsyncProcess: #7, table=usertable, attempt=11/35 failed 2458 ops, last exception: java.io.IOException: Call to sceplus-vm48.almaden.ibm.com/9.1.143.58:60020 failed on local exception: org.apache.hadoop.hbase.ipc.RpcClient$CallTimeoutException: Call id=461, waitTime=2000, rpcTimetout=2000 on sceplus-vm48.almaden.ibm.com,60020,1404809787584, tracking started Tue Jul 08 02:03:05 PDT 2014, retrying after 10058 ms, replay 2458 ops.
14/07/08 02:04:10 INFO client.AsyncProcess: #5, table=usertable, attempt=11/35 SUCCEEDED on sceplus-vm48.almaden.ibm.com,60020,1404809787584, tracking started Tue Jul 08 02:03:05 PDT 2014
14/07/08 02:04:10 INFO client.AsyncProcess: #5, table=usertable, attempt=11/35 SUCCEEDED on sceplus-vm48.almaden.ibm.com,60020,1404809787584, tracking started Tue Jul 08 02:03:02 PDT 2014
14/07/08 02:04:10 INFO client.AsyncProcess: #12, table=usertable, attempt=11/35 SUCCEEDED on sceplus-vm48.almaden.ibm.com,60020,1404809787584, tracking started Tue Jul 08 02:03:05 PDT 2014
2014-07-08 02:04:15:871 80 sec: 433173 operations; 17491.3 current ops/sec; [INSERT AverageLatency(us)=833.09] 
14/07/08 02:04:21 INFO client.AsyncProcess: #14, table=usertable, attempt=12/35 SUCCEEDED on sceplus-vm48.almaden.ibm.com,60020,1404809787584, tracking started Tue Jul 08 02:03:02 PDT 2014
14/07/08 02:04:21 INFO client.AsyncProcess: #19, table=usertable, attempt=12/35 SUCCEEDED on sceplus-vm48.almaden.ibm.com,60020,1404809787584, tracking started Tue Jul 08 02:03:02 PDT 2014
14/07/08 02:04:21 INFO client.AsyncProcess: #18, table=usertable, attempt=12/35 SUCCEEDED on sceplus-vm48.almaden.ibm.com,60020,1404809787584, tracking started Tue Jul 08 02:02:58 PDT 2014
14/07/08 02:04:21 INFO client.AsyncProcess: #12, table=usertable, attempt=12/35 failed 2445 ops, last exception: java.io.IOException: Call to sceplus-vm48.almaden.ibm.com/9.1.143.58:60020 failed on local exception: org.apache.hadoop.hbase.ipc.RpcClient$CallTimeoutException: Call id=586, waitTime=2030, rpcTimetout=2000 on sceplus-vm48.almaden.ibm.com,60020,1404809787584, tracking started Tue Jul 08 02:03:02 PDT 2014, retrying after 20080 ms, replay 2445 ops.
14/07/08 02:04:21 INFO client.AsyncProcess: #2, table=usertable, attempt=12/35 failed 2452 ops, last exception: java.io.IOException: Call to sceplus-vm48.almaden.ibm.com/9.1.143.58:60020 failed on local exception: org.apache.hadoop.hbase.ipc.RpcClient$CallTimeoutException: Call id=586, waitTime=2030, rpcTimetout=2000 on sceplus-vm48.almaden.ibm.com,60020,1404809787584, tracking started Tue Jul 08 02:02:58 PDT 2014, retrying after 20111 ms, replay 2452 ops.
14/07/08 02:04:21 INFO client.AsyncProcess: #4, table=usertable, attempt=12/35 failed 2478 ops, last exception: java.io.IOException: Call to sceplus-vm48.almaden.ibm.com/9.1.143.58:60020 failed on local exception: org.apache.hadoop.hbase.ipc.RpcClient$CallTimeoutException: Call id=586, waitTime=2030, rpcTimetout=2000 on sceplus-vm48.almaden.ibm.com,60020,1404809787584, tracking started Tue Jul 08 02:02:58 PDT 2014, retrying after 20062 ms, replay 2478 ops.
14/07/08 02:04:21 INFO client.AsyncProcess: #20, table=usertable, attempt=12/35 failed 2445 ops, last exception: java.io.IOException: Call to sceplus-vm48.almaden.ibm.com/9.1.143.58:60020 failed on local exception: org.apache.hadoop.hbase.ipc.RpcClient$CallTimeoutException: Call id=590, waitTime=2025, rpcTimetout=2000 on sceplus-vm48.almaden.ibm.com,60020,1404809787584, tracking started Tue Jul 08 02:02:58 PDT 2014, retrying after 20076 ms, replay 2445 ops.
14/07/08 02:04:21 INFO client.AsyncProcess: #13, table=usertable, attempt=12/35 failed 2425 ops, last exception: java.io.IOException: Call to sceplus-vm48.almaden.ibm.com/9.1.143.58:60020 failed on local exception: org.apache.hadoop.hbase.ipc.RpcClient$CallTimeoutException: Call id=590, waitTime=2025, rpcTimetout=2000 on sceplus-vm48.almaden.ibm.com,60020,1404809787584, tracking started Tue Jul 08 02:02:58 PDT 2014, retrying after 20096 ms, replay 2425 ops.
14/07/08 02:04:21 INFO client.AsyncProcess: #10, table=usertable, attempt=12/35 failed 2427 ops, last exception: java.io.IOException: Call to sceplus-vm48.almaden.ibm.com/9.1.143.58:60020 failed on local exception: org.apache.hadoop.hbase.ipc.RpcClient$CallTimeoutException: Call id=590, waitTime=2025, rpcTimetout=2000 on sceplus-vm48.almaden.ibm.com,60020,1404809787584, tracking started Tue Jul 08 02:02:58 PDT 2014, retrying after 20178 ms, replay 2427 ops.
14/07/08 02:04:21 INFO client.AsyncProcess: #8, table=usertable, attempt=12/35 failed 2430 ops, last exception: java.io.IOException: Call to sceplus-vm48.almaden.ibm.com/9.1.143.58:60020 failed on local exception: org.apache.hadoop.hbase.ipc.RpcClient$CallTimeoutException: Call id=590, waitTime=2025, rpcTimetout=2000 on sceplus-vm48.almaden.ibm.com,60020,1404809787584, tracking started Tue Jul 08 02:02:58 PDT 2014, retrying after 20123 ms, replay 2430 ops.
14/07/08 02:04:21 INFO client.AsyncProcess: #17, table=usertable, attempt=12/35 failed 2411 ops, last exception: java.io.IOException: Call to sceplus-vm48.almaden.ibm.com/9.1.143.58:60020 failed on local exception: org.apache.hadoop.hbase.ipc.RpcClient$CallTimeoutException: Call id=590, waitTime=2025, rpcTimetout=2000 on sceplus-vm48.almaden.ibm.com,60020,1404809787584, tracking started Tue Jul 08 02:02:58 PDT 2014, retrying after 20081 ms, replay 2411 ops.
14/07/08 02:04:21 INFO client.AsyncProcess: #6, table=usertable, attempt=12/35 failed 2463 ops, last exception: java.io.IOException: Call to sceplus-vm48.almaden.ibm.com/9.1.143.58:60020 failed on local exception: org.apache.hadoop.hbase.ipc.RpcClient$CallTimeoutException: Call id=590, waitTime=2025, rpcTimetout=2000 on sceplus-vm48.almaden.ibm.com,60020,1404809787584, tracking started Tue Jul 08 02:03:02 PDT 2014, retrying after 20188 ms, replay 2463 ops.
14/07/08 02:04:21 INFO client.AsyncProcess: #15, table=usertable, attempt=12/35 failed 2420 ops, last exception: java.io.IOException: Call to sceplus-vm48.almaden.ibm.com/9.1.143.58:60020 failed on local exception: org.apache.hadoop.hbase.ipc.RpcClient$CallTimeoutException: Call id=590, waitTime=2025, rpcTimetout=2000 on sceplus-vm48.almaden.ibm.com,60020,1404809787584, tracking started Tue Jul 08 02:02:58 PDT 2014, retrying after 20182 ms, replay 2420 ops.
14/07/08 02:04:21 INFO client.AsyncProcess: #14, table=usertable, attempt=12/35 failed 2412 ops, last exception: java.io.IOException: Call to sceplus-vm48.almaden.ibm.com/9.1.143.58:60020 failed on local exception: org.apache.hadoop.hbase.ipc.RpcClient$CallTimeoutException: Call id=590, waitTime=2025, rpcTimetout=2000 on sceplus-vm48.almaden.ibm.com,60020,1404809787584, tracking started Tue Jul 08 02:02:58 PDT 2014, retrying after 20108 ms, replay 2412 ops.
14/07/08 02:04:22 INFO client.AsyncProcess: #7, table=usertable, attempt=12/35 failed 2456 ops, last exception: java.io.IOException: Call to sceplus-vm48.almaden.ibm.com/9.1.143.58:60020 failed on local exception: org.apache.hadoop.hbase.ipc.RpcClient$CallTimeoutException: Call id=598, waitTime=2149, rpcTimetout=2000 on sceplus-vm48.almaden.ibm.com,60020,1404809787584, tracking started Tue Jul 08 02:03:02 PDT 2014, retrying after 20081 ms, replay 2456 ops.
14/07/08 02:04:22 INFO client.AsyncProcess: #3, table=usertable, attempt=12/35 failed 2469 ops, last exception: java.io.IOException: Call to sceplus-vm48.almaden.ibm.com/9.1.143.58:60020 failed on local exception: org.apache.hadoop.hbase.ipc.RpcClient$CallTimeoutException: Call id=598, waitTime=2149, rpcTimetout=2000 on sceplus-vm48.almaden.ibm.com,60020,1404809787584, tracking started Tue Jul 08 02:02:58 PDT 2014, retrying after 20054 ms, replay 2469 ops.
14/07/08 02:04:22 INFO client.AsyncProcess: #16, table=usertable, attempt=12/35 failed 2443 ops, last exception: java.io.IOException: Call to sceplus-vm48.almaden.ibm.com/9.1.143.58:60020 failed on local exception: org.apache.hadoop.hbase.ipc.RpcClient$CallTimeoutException: Call id=598, waitTime=2149, rpcTimetout=2000 on sceplus-vm48.almaden.ibm.com,60020,1404809787584, tracking started Tue Jul 08 02:02:58 PDT 2014, retrying after 20123 ms, replay 2443 ops.
14/07/08 02:04:22 INFO client.AsyncProcess: #2, table=usertable, attempt=12/35 failed 2435 ops, last exception: java.io.IOException: Call to sceplus-vm48.almaden.ibm.com/9.1.143.58:60020 failed on local exception: org.apache.hadoop.hbase.ipc.RpcClient$CallTimeoutException: Call id=598, waitTime=2149, rpcTimetout=2000 on sceplus-vm48.almaden.ibm.com,60020,1404809787584, tracking started Tue Jul 08 02:03:02 PDT 2014, retrying after 20125 ms, replay 2435 ops.
14/07/08 02:04:22 INFO client.AsyncProcess: #11, table=usertable, attempt=12/35 SUCCEEDED on sceplus-vm48.almaden.ibm.com,60020,1404809787584, tracking started Tue Jul 08 02:02:58 PDT 2014
14/07/08 02:04:22 INFO client.AsyncProcess: #12, table=usertable, attempt=12/35 failed 2431 ops, last exception: java.io.IOException: Call to sceplus-vm48.almaden.ibm.com/9.1.143.58:60020 failed on local exception: org.apache.hadoop.hbase.ipc.RpcClient$CallTimeoutException: Call id=598, waitTime=2149, rpcTimetout=2000 on sceplus-vm48.almaden.ibm.com,60020,1404809787584, tracking started Tue Jul 08 02:02:58 PDT 2014, retrying after 20119 ms, replay 2431 ops.
14/07/08 02:04:22 INFO client.AsyncProcess: #6, table=usertable, attempt=12/35 failed 2404 ops, last exception: java.io.IOException: Call to sceplus-vm48.almaden.ibm.com/9.1.143.58:60020 failed on local exception: org.apache.hadoop.hbase.ipc.RpcClient$CallTimeoutException: Call id=598, waitTime=2149, rpcTimetout=2000 on sceplus-vm48.almaden.ibm.com,60020,1404809787584, tracking started Tue Jul 08 02:02:58 PDT 2014, retrying after 20000 ms, replay 2404 ops.
14/07/08 02:04:22 INFO client.AsyncProcess: #9, table=usertable, attempt=12/35 failed 2466 ops, last exception: java.io.IOException: Call to sceplus-vm48.almaden.ibm.com/9.1.143.58:60020 failed on local exception: org.apache.hadoop.hbase.ipc.RpcClient$CallTimeoutException: Call id=608, waitTime=2147, rpcTimetout=2000 on sceplus-vm48.almaden.ibm.com,60020,1404809787584, tracking started Tue Jul 08 02:03:03 PDT 2014, retrying after 20180 ms, replay 2466 ops.
14/07/08 02:04:22 INFO client.AsyncProcess: #7, table=usertable, attempt=12/35 failed 2458 ops, last exception: java.io.IOException: Call to sceplus-vm48.almaden.ibm.com/9.1.143.58:60020 failed on local exception: org.apache.hadoop.hbase.ipc.RpcClient$CallTimeoutException: Call id=608, waitTime=2147, rpcTimetout=2000 on sceplus-vm48.almaden.ibm.com,60020,1404809787584, tracking started Tue Jul 08 02:03:05 PDT 2014, retrying after 20073 ms, replay 2458 ops.
2014-07-08 02:04:25:872 90 sec: 666812 operations; 23361.56 current ops/sec; [INSERT AverageLatency(us)=1178.68] 
2014-07-08 02:04:35:876 100 sec: 1036503 operations; 36958.01 current ops/sec; [UPDATE AverageLatency(us)=817809] [INSERT AverageLatency(us)=121.69] [CLEANUP AverageLatency(us)=818267.5] 
14/07/08 02:04:44 INFO client.AsyncProcess: #4, table=usertable, attempt=13/35 SUCCEEDED on sceplus-vm48.almaden.ibm.com,60020,1404809787584, tracking started Tue Jul 08 02:02:58 PDT 2014
14/07/08 02:04:44 INFO client.AsyncProcess: #20, table=usertable, attempt=13/35 failed 2445 ops, last exception: java.io.IOException: Call to sceplus-vm48.almaden.ibm.com/9.1.143.58:60020 failed on local exception: org.apache.hadoop.hbase.ipc.RpcClient$CallTimeoutException: Call id=903, waitTime=2590, rpcTimetout=2000 on sceplus-vm48.almaden.ibm.com,60020,1404809787584, tracking started Tue Jul 08 02:02:58 PDT 2014, retrying after 20169 ms, replay 2445 ops.
14/07/08 02:04:44 INFO client.AsyncProcess: #12, table=usertable, attempt=13/35 failed 2431 ops, last exception: java.io.IOException: Call to sceplus-vm48.almaden.ibm.com/9.1.143.58:60020 failed on local exception: org.apache.hadoop.hbase.ipc.RpcClient$CallTimeoutException: Call id=903, waitTime=2590, rpcTimetout=2000 on sceplus-vm48.almaden.ibm.com,60020,1404809787584, tracking started Tue Jul 08 02:02:58 PDT 2014, retrying after 20008 ms, replay 2431 ops.
14/07/08 02:04:44 INFO client.AsyncProcess: #7, table=usertable, attempt=13/35 failed 2456 ops, last exception: java.io.IOException: Call to sceplus-vm48.almaden.ibm.com/9.1.143.58:60020 failed on local exception: org.apache.hadoop.hbase.ipc.RpcClient$CallTimeoutException: Call id=903, waitTime=2590, rpcTimetout=2000 on sceplus-vm48.almaden.ibm.com,60020,1404809787584, tracking started Tue Jul 08 02:03:02 PDT 2014, retrying after 20170 ms, replay 2456 ops.
14/07/08 02:04:44 INFO client.AsyncProcess: #16, table=usertable, attempt=13/35 failed 2443 ops, last exception: java.io.IOException: Call to sceplus-vm48.almaden.ibm.com/9.1.143.58:60020 failed on local exception: org.apache.hadoop.hbase.ipc.RpcClient$CallTimeoutException: Call id=903, waitTime=2590, rpcTimetout=2000 on sceplus-vm48.almaden.ibm.com,60020,1404809787584, tracking started Tue Jul 08 02:02:58 PDT 2014, retrying after 20030 ms, replay 2443 ops.
14/07/08 02:04:44 INFO client.AsyncProcess: #13, table=usertable, attempt=13/35 failed 2425 ops, last exception: java.io.IOException: Call to sceplus-vm48.almaden.ibm.com/9.1.143.58:60020 failed on local exception: org.apache.hadoop.hbase.ipc.RpcClient$CallTimeoutException: Call id=903, waitTime=2590, rpcTimetout=2000 on sceplus-vm48.almaden.ibm.com,60020,1404809787584, tracking started Tue Jul 08 02:02:58 PDT 2014, retrying after 20009 ms, replay 2425 ops.
14/07/08 02:04:44 INFO client.AsyncProcess: #6, table=usertable, attempt=13/35 failed 2404 ops, last exception: java.io.IOException: Call to sceplus-vm48.almaden.ibm.com/9.1.143.58:60020 failed on local exception: org.apache.hadoop.hbase.ipc.RpcClient$CallTimeoutException: Call id=903, waitTime=2590, rpcTimetout=2000 on sceplus-vm48.almaden.ibm.com,60020,1404809787584, tracking started Tue Jul 08 02:02:58 PDT 2014, retrying after 20186 ms, replay 2404 ops.
14/07/08 02:04:44 INFO client.AsyncProcess: #17, table=usertable, attempt=13/35 failed 2411 ops, last exception: java.io.IOException: Call to sceplus-vm48.almaden.ibm.com/9.1.143.58:60020 failed on local exception: org.apache.hadoop.hbase.ipc.RpcClient$CallTimeoutException: Call id=903, waitTime=2590, rpcTimetout=2000 on sceplus-vm48.almaden.ibm.com,60020,1404809787584, tracking started Tue Jul 08 02:02:58 PDT 2014, retrying after 20132 ms, replay 2411 ops.
14/07/08 02:04:44 INFO client.AsyncProcess: #3, table=usertable, attempt=13/35 failed 2469 ops, last exception: java.io.IOException: Call to sceplus-vm48.almaden.ibm.com/9.1.143.58:60020 failed on local exception: org.apache.hadoop.hbase.ipc.RpcClient$CallTimeoutException: Call id=903, waitTime=2590, rpcTimetout=2000 on sceplus-vm48.almaden.ibm.com,60020,1404809787584, tracking started Tue Jul 08 02:02:58 PDT 2014, retrying after 20075 ms, replay 2469 ops.
14/07/08 02:04:44 INFO client.AsyncProcess: #15, table=usertable, attempt=13/35 failed 2420 ops, last exception: java.io.IOException: Call to sceplus-vm48.almaden.ibm.com/9.1.143.58:60020 failed on local exception: org.apache.hadoop.hbase.ipc.RpcClient$CallTimeoutException: Call id=903, waitTime=2590, rpcTimetout=2000 on sceplus-vm48.almaden.ibm.com,60020,1404809787584, tracking started Tue Jul 08 02:02:58 PDT 2014, retrying after 20139 ms, replay 2420 ops.
14/07/08 02:04:44 INFO client.AsyncProcess: #12, table=usertable, attempt=13/35 failed 2445 ops, last exception: java.io.IOException: Call to sceplus-vm48.almaden.ibm.com/9.1.143.58:60020 failed on local exception: org.apache.hadoop.hbase.ipc.RpcClient$CallTimeoutException: Call id=903, waitTime=2590, rpcTimetout=2000 on sceplus-vm48.almaden.ibm.com,60020,1404809787584, tracking started Tue Jul 08 02:03:02 PDT 2014, retrying after 20136 ms, replay 2445 ops.
14/07/08 02:04:44 INFO client.AsyncProcess: #2, table=usertable, attempt=13/35 failed 2435 ops, last exception: java.io.IOException: Call to sceplus-vm48.almaden.ibm.com/9.1.143.58:60020 failed on local exception: org.apache.hadoop.hbase.ipc.RpcClient$CallTimeoutException: Call id=903, waitTime=2590, rpcTimetout=2000 on sceplus-vm48.almaden.ibm.com,60020,1404809787584, tracking started Tue Jul 08 02:03:02 PDT 2014, retrying after 20126 ms, replay 2435 ops.
14/07/08 02:04:44 INFO client.AsyncProcess: #7, table=usertable, attempt=13/35 failed 2458 ops, last exception: java.io.IOException: Call to sceplus-vm48.almaden.ibm.com/9.1.143.58:60020 failed on local exception: org.apache.hadoop.hbase.ipc.RpcClient$CallTimeoutException: Call id=903, waitTime=2590, rpcTimetout=2000 on sceplus-vm48.almaden.ibm.com,60020,1404809787584, tracking started Tue Jul 08 02:03:05 PDT 2014, retrying after 20176 ms, replay 2458 ops.
14/07/08 02:04:44 INFO client.AsyncProcess: #14, table=usertable, attempt=13/35 failed 2412 ops, last exception: java.io.IOException: Call to sceplus-vm48.almaden.ibm.com/9.1.143.58:60020 failed on local exception: org.apache.hadoop.hbase.ipc.RpcClient$CallTimeoutException: Call id=903, waitTime=2590, rpcTimetout=2000 on sceplus-vm48.almaden.ibm.com,60020,1404809787584, tracking started Tue Jul 08 02:02:58 PDT 2014, retrying after 20178 ms, replay 2412 ops.
14/07/08 02:04:44 INFO client.AsyncProcess: #6, table=usertable, attempt=13/35 failed 2463 ops, last exception: java.io.IOException: Call to sceplus-vm48.almaden.ibm.com/9.1.143.58:60020 failed on local exception: org.apache.hadoop.hbase.ipc.RpcClient$CallTimeoutException: Call id=903, waitTime=2590, rpcTimetout=2000 on sceplus-vm48.almaden.ibm.com,60020,1404809787584, tracking started Tue Jul 08 02:03:02 PDT 2014, retrying after 20136 ms, replay 2463 ops.
14/07/08 02:04:44 INFO client.AsyncProcess: #2, table=usertable, attempt=13/35 failed 2452 ops, last exception: java.io.IOException: Call to sceplus-vm48.almaden.ibm.com/9.1.143.58:60020 failed on local exception: org.apache.hadoop.hbase.ipc.RpcClient$CallTimeoutException: Call id=903, waitTime=2590, rpcTimetout=2000 on sceplus-vm48.almaden.ibm.com,60020,1404809787584, tracking started Tue Jul 08 02:02:58 PDT 2014, retrying after 20182 ms, replay 2452 ops.
14/07/08 02:04:44 INFO client.AsyncProcess: #8, table=usertable, attempt=13/35 failed 2430 ops, last exception: java.io.IOException: Call to sceplus-vm48.almaden.ibm.com/9.1.143.58:60020 failed on local exception: org.apache.hadoop.hbase.ipc.RpcClient$CallTimeoutException: Call id=903, waitTime=2590, rpcTimetout=2000 on sceplus-vm48.almaden.ibm.com,60020,1404809787584, tracking started Tue Jul 08 02:02:58 PDT 2014, retrying after 20136 ms, replay 2430 ops.
14/07/08 02:04:44 INFO client.AsyncProcess: #10, table=usertable, attempt=13/35 failed 2427 ops, last exception: java.io.IOException: Call to sceplus-vm48.almaden.ibm.com/9.1.143.58:60020 failed on local exception: org.apache.hadoop.hbase.ipc.RpcClient$CallTimeoutException: Call id=903, waitTime=2590, rpcTimetout=2000 on sceplus-vm48.almaden.ibm.com,60020,1404809787584, tracking started Tue Jul 08 02:02:58 PDT 2014, retrying after 20139 ms, replay 2427 ops.
14/07/08 02:04:44 INFO client.AsyncProcess: #9, table=usertable, attempt=13/35 failed 2466 ops, last exception: java.io.IOException: Call to sceplus-vm48.almaden.ibm.com/9.1.143.58:60020 failed on local exception: org.apache.hadoop.hbase.ipc.RpcClient$CallTimeoutException: Call id=924, waitTime=2034, rpcTimetout=2000 on sceplus-vm48.almaden.ibm.com,60020,1404809787584, tracking started Tue Jul 08 02:03:03 PDT 2014, retrying after 20188 ms, replay 2466 ops.
2014-07-08 02:04:45:877 110 sec: 1223166 operations; 18662.57 current ops/sec;  [INSERT AverageLatency(us)=713.66]  
2014-07-08 02:04:55:877 120 sec: 1486142 operations; 26297.6 current ops/sec;  [INSERT AverageLatency(us)=159.47]  
2014-07-08 02:05:05:878 130 sec: 1658537 operations; 17237.78 current ops/sec; [UPDATE AverageLatency(us)=584504] [INSERT AverageLatency(us)=89.99] [CLEANUP AverageLatency(us)=584524] 
14/07/08 02:05:06 INFO client.AsyncProcess: #16, table=usertable, attempt=14/35 SUCCEEDED on sceplus-vm48.almaden.ibm.com,60020,1404809787584, tracking started Tue Jul 08 02:02:58 PDT 2014
14/07/08 02:05:06 INFO client.AsyncProcess: #12, table=usertable, attempt=14/35 SUCCEEDED on sceplus-vm48.almaden.ibm.com,60020,1404809787584, tracking started Tue Jul 08 02:02:58 PDT 2014
14/07/08 02:05:06 INFO client.AsyncProcess: #13, table=usertable, attempt=14/35 SUCCEEDED on sceplus-vm48.almaden.ibm.com,60020,1404809787584, tracking started Tue Jul 08 02:02:58 PDT 2014
14/07/08 02:05:06 INFO client.AsyncProcess: #3, table=usertable, attempt=14/35 SUCCEEDED on sceplus-vm48.almaden.ibm.com,60020,1404809787584, tracking started Tue Jul 08 02:02:58 PDT 2014
14/07/08 02:05:06 INFO client.AsyncProcess: #2, table=usertable, attempt=14/35 SUCCEEDED on sceplus-vm48.almaden.ibm.com,60020,1404809787584, tracking started Tue Jul 08 02:03:02 PDT 2014
14/07/08 02:05:06 INFO client.AsyncProcess: #6, table=usertable, attempt=14/35 SUCCEEDED on sceplus-vm48.almaden.ibm.com,60020,1404809787584, tracking started Tue Jul 08 02:03:02 PDT 2014
14/07/08 02:05:06 INFO client.AsyncProcess: #12, table=usertable, attempt=14/35 SUCCEEDED on sceplus-vm48.almaden.ibm.com,60020,1404809787584, tracking started Tue Jul 08 02:03:02 PDT 2014
14/07/08 02:05:06 INFO client.AsyncProcess: #15, table=usertable, attempt=14/35 failed 2420 ops, last exception: java.io.IOException: Call to sceplus-vm48.almaden.ibm.com/9.1.143.58:60020 failed on local exception: org.apache.hadoop.hbase.ipc.RpcClient$CallTimeoutException: Call id=1151, waitTime=2018, rpcTimetout=2000 on sceplus-vm48.almaden.ibm.com,60020,1404809787584, tracking started Tue Jul 08 02:02:58 PDT 2014, retrying after 20130 ms, replay 2420 ops.
14/07/08 02:05:06 INFO client.AsyncProcess: #8, table=usertable, attempt=14/35 failed 2430 ops, last exception: java.io.IOException: Call to sceplus-vm48.almaden.ibm.com/9.1.143.58:60020 failed on local exception: org.apache.hadoop.hbase.ipc.RpcClient$CallTimeoutException: Call id=1151, waitTime=2018, rpcTimetout=2000 on sceplus-vm48.almaden.ibm.com,60020,1404809787584, tracking started Tue Jul 08 02:02:58 PDT 2014, retrying after 20196 ms, replay 2430 ops.
14/07/08 02:05:06 INFO client.AsyncProcess: #10, table=usertable, attempt=14/35 failed 2427 ops, last exception: java.io.IOException: Call to sceplus-vm48.almaden.ibm.com/9.1.143.58:60020 failed on local exception: org.apache.hadoop.hbase.ipc.RpcClient$CallTimeoutException: Call id=1151, waitTime=2018, rpcTimetout=2000 on sceplus-vm48.almaden.ibm.com,60020,1404809787584, tracking started Tue Jul 08 02:02:58 PDT 2014, retrying after 20123 ms, replay 2427 ops.
14/07/08 02:05:06 INFO client.AsyncProcess: #17, table=usertable, attempt=14/35 failed 2411 ops, last exception: java.io.IOException: Call to sceplus-vm48.almaden.ibm.com/9.1.143.58:60020 failed on local exception: org.apache.hadoop.hbase.ipc.RpcClient$CallTimeoutException: Call id=1151, waitTime=2018, rpcTimetout=2000 on sceplus-vm48.almaden.ibm.com,60020,1404809787584, tracking started Tue Jul 08 02:02:58 PDT 2014, retrying after 20065 ms, replay 2411 ops.
14/07/08 02:05:06 INFO client.AsyncProcess: #7, table=usertable, attempt=14/35 failed 2456 ops, last exception: java.io.IOException: Call to sceplus-vm48.almaden.ibm.com/9.1.143.58:60020 failed on local exception: org.apache.hadoop.hbase.ipc.RpcClient$CallTimeoutException: Call id=1157, waitTime=2069, rpcTimetout=2000 on sceplus-vm48.almaden.ibm.com,60020,1404809787584, tracking started Tue Jul 08 02:03:02 PDT 2014, retrying after 20124 ms, replay 2456 ops.
14/07/08 02:05:06 INFO client.AsyncProcess: #2, table=usertable, attempt=14/35 failed 2452 ops, last exception: java.io.IOException: Call to sceplus-vm48.almaden.ibm.com/9.1.143.58:60020 failed on local exception: org.apache.hadoop.hbase.ipc.RpcClient$CallTimeoutException: Call id=1157, waitTime=2069, rpcTimetout=2000 on sceplus-vm48.almaden.ibm.com,60020,1404809787584, tracking started Tue Jul 08 02:02:58 PDT 2014, retrying after 20013 ms, replay 2452 ops.
14/07/08 02:05:06 INFO client.AsyncProcess: #7, table=usertable, attempt=14/35 failed 2458 ops, last exception: java.io.IOException: Call to sceplus-vm48.almaden.ibm.com/9.1.143.58:60020 failed on local exception: org.apache.hadoop.hbase.ipc.RpcClient$CallTimeoutException: Call id=1157, waitTime=2069, rpcTimetout=2000 on sceplus-vm48.almaden.ibm.com,60020,1404809787584, tracking started Tue Jul 08 02:03:05 PDT 2014, retrying after 20006 ms, replay 2458 ops.
14/07/08 02:05:06 INFO client.AsyncProcess: #9, table=usertable, attempt=14/35 SUCCEEDED on sceplus-vm48.almaden.ibm.com,60020,1404809787584, tracking started Tue Jul 08 02:03:03 PDT 2014
14/07/08 02:05:06 INFO client.AsyncProcess: #6, table=usertable, attempt=14/35 failed 2404 ops, last exception: java.io.IOException: Call to sceplus-vm48.almaden.ibm.com/9.1.143.58:60020 failed on local exception: org.apache.hadoop.hbase.ipc.RpcClient$CallTimeoutException: Call id=1157, waitTime=2069, rpcTimetout=2000 on sceplus-vm48.almaden.ibm.com,60020,1404809787584, tracking started Tue Jul 08 02:02:58 PDT 2014, retrying after 20162 ms, replay 2404 ops.
14/07/08 02:05:06 INFO client.AsyncProcess: #14, table=usertable, attempt=14/35 failed 2412 ops, last exception: java.io.IOException: Call to sceplus-vm48.almaden.ibm.com/9.1.143.58:60020 failed on local exception: org.apache.hadoop.hbase.ipc.RpcClient$CallTimeoutException: Call id=1157, waitTime=2069, rpcTimetout=2000 on sceplus-vm48.almaden.ibm.com,60020,1404809787584, tracking started Tue Jul 08 02:02:58 PDT 2014, retrying after 20015 ms, replay 2412 ops.
14/07/08 02:05:06 INFO client.AsyncProcess: #20, table=usertable, attempt=14/35 failed 2445 ops, last exception: java.io.IOException: Call to sceplus-vm48.almaden.ibm.com/9.1.143.58:60020 failed on local exception: org.apache.hadoop.hbase.ipc.RpcClient$CallTimeoutException: Call id=1157, waitTime=2069, rpcTimetout=2000 on sceplus-vm48.almaden.ibm.com,60020,1404809787584, tracking started Tue Jul 08 02:02:58 PDT 2014, retrying after 20110 ms, replay 2445 ops.
2014-07-08 02:05:15:879 140 sec: 1936714 operations; 27814.92 current ops/sec; [UPDATE AverageLatency(us)=1239111] [INSERT AverageLatency(us)=2429.48] [CLEANUP AverageLatency(us)=1239131] 
2014-07-08 02:05:25:880 150 sec: 2293610 operations; 35686.03 current ops/sec;  [INSERT AverageLatency(us)=135.94]  
14/07/08 02:05:28 INFO client.AsyncProcess: #7, table=usertable, attempt=15/35 SUCCEEDED on sceplus-vm48.almaden.ibm.com,60020,1404809787584, tracking started Tue Jul 08 02:03:02 PDT 2014
14/07/08 02:05:28 INFO client.AsyncProcess: #7, table=usertable, attempt=15/35 failed 2458 ops, last exception: java.io.IOException: Call to sceplus-vm48.almaden.ibm.com/9.1.143.58:60020 failed on local exception: org.apache.hadoop.hbase.ipc.RpcClient$CallTimeoutException: Call id=1481, waitTime=2141, rpcTimetout=2000 on sceplus-vm48.almaden.ibm.com,60020,1404809787584, tracking started Tue Jul 08 02:03:05 PDT 2014, retrying after 20172 ms, replay 2458 ops.
14/07/08 02:05:28 INFO client.AsyncProcess: #15, table=usertable, attempt=15/35 failed 2420 ops, last exception: java.io.IOException: Call to sceplus-vm48.almaden.ibm.com/9.1.143.58:60020 failed on local exception: org.apache.hadoop.hbase.ipc.RpcClient$CallTimeoutException: Call id=1481, waitTime=2141, rpcTimetout=2000 on sceplus-vm48.almaden.ibm.com,60020,1404809787584, tracking started Tue Jul 08 02:02:58 PDT 2014, retrying after 20012 ms, replay 2420 ops.
14/07/08 02:05:28 INFO client.AsyncProcess: #6, table=usertable, attempt=15/35 SUCCEEDED on sceplus-vm48.almaden.ibm.com,60020,1404809787584, tracking started Tue Jul 08 02:02:58 PDT 2014
14/07/08 02:05:28 INFO client.AsyncProcess: #17, table=usertable, attempt=15/35 SUCCEEDED on sceplus-vm48.almaden.ibm.com,60020,1404809787584, tracking started Tue Jul 08 02:02:58 PDT 2014
14/07/08 02:05:28 INFO client.AsyncProcess: #2, table=usertable, attempt=15/35 failed 2452 ops, last exception: java.io.IOException: Call to sceplus-vm48.almaden.ibm.com/9.1.143.58:60020 failed on local exception: org.apache.hadoop.hbase.ipc.RpcClient$CallTimeoutException: Call id=1487, waitTime=2018, rpcTimetout=2000 on sceplus-vm48.almaden.ibm.com,60020,1404809787584, tracking started Tue Jul 08 02:02:58 PDT 2014, retrying after 20071 ms, replay 2452 ops.
14/07/08 02:05:28 INFO client.AsyncProcess: #20, table=usertable, attempt=15/35 SUCCEEDED on sceplus-vm48.almaden.ibm.com,60020,1404809787584, tracking started Tue Jul 08 02:02:58 PDT 2014
14/07/08 02:05:28 INFO client.AsyncProcess: #8, table=usertable, attempt=15/35 failed 2430 ops, last exception: java.io.IOException: Call to sceplus-vm48.almaden.ibm.com/9.1.143.58:60020 failed on local exception: org.apache.hadoop.hbase.ipc.RpcClient$CallTimeoutException: Call id=1489, waitTime=2024, rpcTimetout=2000 on sceplus-vm48.almaden.ibm.com,60020,1404809787584, tracking started Tue Jul 08 02:02:58 PDT 2014, retrying after 20079 ms, replay 2430 ops.
14/07/08 02:05:29 INFO client.AsyncProcess: #10, table=usertable, attempt=15/35 SUCCEEDED on sceplus-vm48.almaden.ibm.com,60020,1404809787584, tracking started Tue Jul 08 02:02:58 PDT 2014
14/07/08 02:05:29 INFO client.AsyncProcess: #14, table=usertable, attempt=15/35 SUCCEEDED on sceplus-vm48.almaden.ibm.com,60020,1404809787584, tracking started Tue Jul 08 02:02:58 PDT 2014
2014-07-08 02:05:36:750 160 sec: 2504930 operations; 19444.24 current ops/sec;  [INSERT AverageLatency(us)=3707.49]  
2014-07-08 02:05:46:750 170 sec: 2504930 operations; 0 current ops/sec;    
14/07/08 02:05:50 INFO client.AsyncProcess: #15, table=usertable, attempt=16/35 SUCCEEDED on sceplus-vm48.almaden.ibm.com,60020,1404809787584, tracking started Tue Jul 08 02:02:58 PDT 2014
14/07/08 02:05:50 INFO client.AsyncProcess: #8, table=usertable, attempt=16/35 SUCCEEDED on sceplus-vm48.almaden.ibm.com,60020,1404809787584, tracking started Tue Jul 08 02:02:58 PDT 2014
14/07/08 02:05:50 INFO client.AsyncProcess: #7, table=usertable, attempt=16/35 SUCCEEDED on sceplus-vm48.almaden.ibm.com,60020,1404809787584, tracking started Tue Jul 08 02:03:05 PDT 2014
14/07/08 02:05:50 INFO client.AsyncProcess: #2, table=usertable, attempt=16/35 SUCCEEDED on sceplus-vm48.almaden.ibm.com,60020,1404809787584, tracking started Tue Jul 08 02:02:58 PDT 2014
2014-07-08 02:05:56:751 180 sec: 2609288 operations; 10434.76 current ops/sec;  [INSERT AverageLatency(us)=6666.34]  
2014-07-08 02:06:06:752 190 sec: 2852434 operations; 24312.17 current ops/sec;  [INSERT AverageLatency(us)=164.05]  
2014-07-08 02:06:16:753 200 sec: 3077842 operations; 22540.8 current ops/sec;  [INSERT AverageLatency(us)=172.51]  
14/07/08 02:06:23 INFO client.AsyncProcess: #20, table=usertable, attempt=10/35 failed 2412 ops, last exception: java.io.IOException: Call to sceplus-vm48.almaden.ibm.com/9.1.143.58:60020 failed on local exception: org.apache.hadoop.hbase.ipc.RpcClient$CallTimeoutException: Call id=2199, waitTime=2317, rpcTimetout=2000 on sceplus-vm48.almaden.ibm.com,60020,1404809787584, tracking started Tue Jul 08 02:05:30 PDT 2014, retrying after 10056 ms, replay 2412 ops.
14/07/08 02:06:23 INFO client.AsyncProcess: #10, table=usertable, attempt=10/35 failed 2493 ops, last exception: java.io.IOException: Call to sceplus-vm48.almaden.ibm.com/9.1.143.58:60020 failed on local exception: org.apache.hadoop.hbase.ipc.RpcClient$CallTimeoutException: Call id=2199, waitTime=2317, rpcTimetout=2000 on sceplus-vm48.almaden.ibm.com,60020,1404809787584, tracking started Tue Jul 08 02:05:30 PDT 2014, retrying after 10051 ms, replay 2493 ops.
14/07/08 02:06:23 INFO client.AsyncProcess: #14, table=usertable, attempt=10/35 failed 2421 ops, last exception: java.io.IOException: Call to sceplus-vm48.almaden.ibm.com/9.1.143.58:60020 failed on local exception: org.apache.hadoop.hbase.ipc.RpcClient$CallTimeoutException: Call id=2199, waitTime=2317, rpcTimetout=2000 on sceplus-vm48.almaden.ibm.com,60020,1404809787584, tracking started Tue Jul 08 02:05:30 PDT 2014, retrying after 10072 ms, replay 2421 ops.
14/07/08 02:06:23 INFO client.AsyncProcess: #3, table=usertable, attempt=10/35 failed 2440 ops, last exception: java.io.IOException: Call to sceplus-vm48.almaden.ibm.com/9.1.143.58:60020 failed on local exception: org.apache.hadoop.hbase.ipc.RpcClient$CallTimeoutException: Call id=2199, waitTime=2317, rpcTimetout=2000 on sceplus-vm48.almaden.ibm.com,60020,1404809787584, tracking started Tue Jul 08 02:05:30 PDT 2014, retrying after 10010 ms, replay 2440 ops.
14/07/08 02:06:23 INFO client.AsyncProcess: #3, table=usertable, attempt=10/35 failed 2463 ops, last exception: java.io.IOException: Call to sceplus-vm48.almaden.ibm.com/9.1.143.58:60020 failed on local exception: org.apache.hadoop.hbase.ipc.RpcClient$CallTimeoutException: Call id=2199, waitTime=2317, rpcTimetout=2000 on sceplus-vm48.almaden.ibm.com,60020,1404809787584, tracking started Tue Jul 08 02:05:31 PDT 2014, retrying after 10056 ms, replay 2463 ops.
14/07/08 02:06:23 INFO client.AsyncProcess: #9, table=usertable, attempt=10/35 failed 2440 ops, last exception: java.io.IOException: Call to sceplus-vm48.almaden.ibm.com/9.1.143.58:60020 failed on local exception: org.apache.hadoop.hbase.ipc.RpcClient$CallTimeoutException: Call id=2199, waitTime=2317, rpcTimetout=2000 on sceplus-vm48.almaden.ibm.com,60020,1404809787584, tracking started Tue Jul 08 02:05:30 PDT 2014, retrying after 10093 ms, replay 2440 ops.
14/07/08 02:06:23 INFO client.AsyncProcess: #10, table=usertable, attempt=10/35 failed 2364 ops, last exception: java.io.IOException: Call to sceplus-vm48.almaden.ibm.com/9.1.143.58:60020 failed on local exception: org.apache.hadoop.hbase.ipc.RpcClient$CallTimeoutException: Call id=2199, waitTime=2317, rpcTimetout=2000 on sceplus-vm48.almaden.ibm.com,60020,1404809787584, tracking started Tue Jul 08 02:05:31 PDT 2014, retrying after 10075 ms, replay 2364 ops.
14/07/08 02:06:23 INFO client.AsyncProcess: #12, table=usertable, attempt=10/35 failed 2400 ops, last exception: java.io.IOException: Call to sceplus-vm48.almaden.ibm.com/9.1.143.58:60020 failed on local exception: org.apache.hadoop.hbase.ipc.RpcClient$CallTimeoutException: Call id=2199, waitTime=2317, rpcTimetout=2000 on sceplus-vm48.almaden.ibm.com,60020,1404809787584, tracking started Tue Jul 08 02:05:31 PDT 2014, retrying after 10095 ms, replay 2400 ops.
14/07/08 02:06:23 INFO client.AsyncProcess: #17, table=usertable, attempt=10/35 failed 2465 ops, last exception: java.io.IOException: Call to sceplus-vm48.almaden.ibm.com/9.1.143.58:60020 failed on local exception: org.apache.hadoop.hbase.ipc.RpcClient$CallTimeoutException: Call id=2199, waitTime=2317, rpcTimetout=2000 on sceplus-vm48.almaden.ibm.com,60020,1404809787584, tracking started Tue Jul 08 02:05:30 PDT 2014, retrying after 10082 ms, replay 2465 ops.
14/07/08 02:06:23 INFO client.AsyncProcess: #17, table=usertable, attempt=10/35 failed 2410 ops, last exception: java.io.IOException: Call to sceplus-vm48.almaden.ibm.com/9.1.143.58:60020 failed on local exception: org.apache.hadoop.hbase.ipc.RpcClient$CallTimeoutException: Call id=2199, waitTime=2317, rpcTimetout=2000 on sceplus-vm48.almaden.ibm.com,60020,1404809787584, tracking started Tue Jul 08 02:05:30 PDT 2014, retrying after 10002 ms, replay 2410 ops.
14/07/08 02:06:23 INFO client.AsyncProcess: #6, table=usertable, attempt=10/35 failed 2419 ops, last exception: java.io.IOException: Call to sceplus-vm48.almaden.ibm.com/9.1.143.58:60020 failed on local exception: org.apache.hadoop.hbase.ipc.RpcClient$CallTimeoutException: Call id=2199, waitTime=2317, rpcTimetout=2000 on sceplus-vm48.almaden.ibm.com,60020,1404809787584, tracking started Tue Jul 08 02:05:30 PDT 2014, retrying after 10055 ms, replay 2419 ops.
14/07/08 02:06:23 INFO client.AsyncProcess: #14, table=usertable, attempt=10/35 failed 2435 ops, last exception: java.io.IOException: Call to sceplus-vm48.almaden.ibm.com/9.1.143.58:60020 failed on local exception: org.apache.hadoop.hbase.ipc.RpcClient$CallTimeoutException: Call id=2199, waitTime=2317, rpcTimetout=2000 on sceplus-vm48.almaden.ibm.com,60020,1404809787584, tracking started Tue Jul 08 02:05:29 PDT 2014, retrying after 10001 ms, replay 2435 ops.
14/07/08 02:06:23 INFO client.AsyncProcess: #10, table=usertable, attempt=10/35 failed 2468 ops, last exception: java.io.IOException: Call to sceplus-vm48.almaden.ibm.com/9.1.143.58:60020 failed on local exception: org.apache.hadoop.hbase.ipc.RpcClient$CallTimeoutException: Call id=2199, waitTime=2317, rpcTimetout=2000 on sceplus-vm48.almaden.ibm.com,60020,1404809787584, tracking started Tue Jul 08 02:05:31 PDT 2014, retrying after 10005 ms, replay 2468 ops.
14/07/08 02:06:23 INFO client.AsyncProcess: #16, table=usertable, attempt=10/35 failed 2402 ops, last exception: java.io.IOException: Call to sceplus-vm48.almaden.ibm.com/9.1.143.58:60020 failed on local exception: org.apache.hadoop.hbase.ipc.RpcClient$CallTimeoutException: Call id=2199, waitTime=2317, rpcTimetout=2000 on sceplus-vm48.almaden.ibm.com,60020,1404809787584, tracking started Tue Jul 08 02:05:30 PDT 2014, retrying after 10057 ms, replay 2402 ops.
14/07/08 02:06:23 INFO client.AsyncProcess: #6, table=usertable, attempt=10/35 failed 2405 ops, last exception: java.io.IOException: Call to sceplus-vm48.almaden.ibm.com/9.1.143.58:60020 failed on local exception: org.apache.hadoop.hbase.ipc.RpcClient$CallTimeoutException: Call id=2199, waitTime=2317, rpcTimetout=2000 on sceplus-vm48.almaden.ibm.com,60020,1404809787584, tracking started Tue Jul 08 02:05:30 PDT 2014, retrying after 10044 ms, replay 2405 ops.
14/07/08 02:06:23 INFO client.AsyncProcess: #20, table=usertable, attempt=10/35 failed 2462 ops, last exception: java.io.IOException: Call to sceplus-vm48.almaden.ibm.com/9.1.143.58:60020 failed on local exception: org.apache.hadoop.hbase.ipc.RpcClient$CallTimeoutException: Call id=2199, waitTime=2317, rpcTimetout=2000 on sceplus-vm48.almaden.ibm.com,60020,1404809787584, tracking started Tue Jul 08 02:05:30 PDT 2014, retrying after 10008 ms, replay 2462 ops.
14/07/08 02:06:23 INFO client.AsyncProcess: #13, table=usertable, attempt=10/35 failed 2414 ops, last exception: java.io.IOException: Call to sceplus-vm48.almaden.ibm.com/9.1.143.58:60020 failed on local exception: org.apache.hadoop.hbase.ipc.RpcClient$CallTimeoutException: Call id=2199, waitTime=2317, rpcTimetout=2000 on sceplus-vm48.almaden.ibm.com,60020,1404809787584, tracking started Tue Jul 08 02:05:31 PDT 2014, retrying after 10049 ms, replay 2414 ops.
14/07/08 02:06:24 INFO client.AsyncProcess: #3, table=usertable, attempt=10/35 failed 2426 ops, last exception: java.io.IOException: Call to sceplus-vm48.almaden.ibm.com/9.1.143.58:60020 failed on local exception: org.apache.hadoop.hbase.ipc.RpcClient$CallTimeoutException: Call id=2220, waitTime=2739, rpcTimetout=2000 on sceplus-vm48.almaden.ibm.com,60020,1404809787584, tracking started Tue Jul 08 02:05:32 PDT 2014, retrying after 10007 ms, replay 2426 ops.
14/07/08 02:06:24 INFO client.AsyncProcess: #3, table=usertable, attempt=10/35 failed 2450 ops, last exception: java.io.IOException: Call to sceplus-vm48.almaden.ibm.com/9.1.143.58:60020 failed on local exception: org.apache.hadoop.hbase.ipc.RpcClient$CallTimeoutException: Call id=2220, waitTime=2739, rpcTimetout=2000 on sceplus-vm48.almaden.ibm.com,60020,1404809787584, tracking started Tue Jul 08 02:05:30 PDT 2014, retrying after 10041 ms, replay 2450 ops.
14/07/08 02:06:25 INFO client.AsyncProcess: #10, table=usertable, attempt=10/35 failed 2464 ops, last exception: java.io.IOException: Call to sceplus-vm48.almaden.ibm.com/9.1.143.58:60020 failed on local exception: org.apache.hadoop.hbase.ipc.RpcClient$CallTimeoutException: Call id=2232, waitTime=2247, rpcTimetout=2000 on sceplus-vm48.almaden.ibm.com,60020,1404809787584, tracking started Tue Jul 08 02:05:32 PDT 2014, retrying after 10059 ms, replay 2464 ops.
2014-07-08 02:06:26:753 210 sec: 3312642 operations; 23477.65 current ops/sec;  [INSERT AverageLatency(us)=138.97]  
14/07/08 02:06:35 INFO client.AsyncProcess: #17, table=usertable, attempt=11/35 SUCCEEDED on sceplus-vm48.almaden.ibm.com,60020,1404809787584, tracking started Tue Jul 08 02:05:30 PDT 2014
14/07/08 02:06:35 INFO client.AsyncProcess: #3, table=usertable, attempt=11/35 SUCCEEDED on sceplus-vm48.almaden.ibm.com,60020,1404809787584, tracking started Tue Jul 08 02:05:30 PDT 2014
14/07/08 02:06:35 INFO client.AsyncProcess: #20, table=usertable, attempt=11/35 SUCCEEDED on sceplus-vm48.almaden.ibm.com,60020,1404809787584, tracking started Tue Jul 08 02:05:30 PDT 2014
14/07/08 02:06:35 INFO client.AsyncProcess: #14, table=usertable, attempt=11/35 SUCCEEDED on sceplus-vm48.almaden.ibm.com,60020,1404809787584, tracking started Tue Jul 08 02:05:29 PDT 2014
14/07/08 02:06:35 INFO client.AsyncProcess: #10, table=usertable, attempt=11/35 SUCCEEDED on sceplus-vm48.almaden.ibm.com,60020,1404809787584, tracking started Tue Jul 08 02:05:31 PDT 2014
14/07/08 02:06:35 INFO client.AsyncProcess: #17, table=usertable, attempt=11/35 SUCCEEDED on sceplus-vm48.almaden.ibm.com,60020,1404809787584, tracking started Tue Jul 08 02:05:30 PDT 2014
14/07/08 02:06:35 INFO client.AsyncProcess: #9, table=usertable, attempt=11/35 SUCCEEDED on sceplus-vm48.almaden.ibm.com,60020,1404809787584, tracking started Tue Jul 08 02:05:30 PDT 2014
14/07/08 02:06:35 INFO client.AsyncProcess: #14, table=usertable, attempt=11/35 SUCCEEDED on sceplus-vm48.almaden.ibm.com,60020,1404809787584, tracking started Tue Jul 08 02:05:30 PDT 2014
14/07/08 02:06:35 INFO client.AsyncProcess: #12, table=usertable, attempt=11/35 SUCCEEDED on sceplus-vm48.almaden.ibm.com,60020,1404809787584, tracking started Tue Jul 08 02:05:31 PDT 2014
14/07/08 02:06:35 INFO client.AsyncProcess: #10, table=usertable, attempt=11/35 SUCCEEDED on sceplus-vm48.almaden.ibm.com,60020,1404809787584, tracking started Tue Jul 08 02:05:31 PDT 2014
14/07/08 02:06:35 INFO client.AsyncProcess: #6, table=usertable, attempt=11/35 SUCCEEDED on sceplus-vm48.almaden.ibm.com,60020,1404809787584, tracking started Tue Jul 08 02:05:30 PDT 2014
14/07/08 02:06:35 INFO client.AsyncProcess: #10, table=usertable, attempt=11/35 SUCCEEDED on sceplus-vm48.almaden.ibm.com,60020,1404809787584, tracking started Tue Jul 08 02:05:30 PDT 2014
14/07/08 02:06:35 INFO client.AsyncProcess: #13, table=usertable, attempt=11/35 SUCCEEDED on sceplus-vm48.almaden.ibm.com,60020,1404809787584, tracking started Tue Jul 08 02:05:31 PDT 2014
14/07/08 02:06:35 INFO client.AsyncProcess: #3, table=usertable, attempt=11/35 SUCCEEDED on sceplus-vm48.almaden.ibm.com,60020,1404809787584, tracking started Tue Jul 08 02:05:31 PDT 2014
14/07/08 02:06:35 INFO client.AsyncProcess: #6, table=usertable, attempt=11/35 SUCCEEDED on sceplus-vm48.almaden.ibm.com,60020,1404809787584, tracking started Tue Jul 08 02:05:30 PDT 2014
14/07/08 02:06:35 INFO client.AsyncProcess: #3, table=usertable, attempt=11/35 SUCCEEDED on sceplus-vm48.almaden.ibm.com,60020,1404809787584, tracking started Tue Jul 08 02:05:30 PDT 2014
14/07/08 02:06:35 INFO client.AsyncProcess: #20, table=usertable, attempt=11/35 SUCCEEDED on sceplus-vm48.almaden.ibm.com,60020,1404809787584, tracking started Tue Jul 08 02:05:30 PDT 2014
14/07/08 02:06:35 INFO client.AsyncProcess: #16, table=usertable, attempt=11/35 SUCCEEDED on sceplus-vm48.almaden.ibm.com,60020,1404809787584, tracking started Tue Jul 08 02:05:30 PDT 2014
14/07/08 02:06:35 INFO client.AsyncProcess: #3, table=usertable, attempt=11/35 SUCCEEDED on sceplus-vm48.almaden.ibm.com,60020,1404809787584, tracking started Tue Jul 08 02:05:32 PDT 2014
14/07/08 02:06:36 INFO client.AsyncProcess: #10, table=usertable, attempt=11/35 SUCCEEDED on sceplus-vm48.almaden.ibm.com,60020,1404809787584, tracking started Tue Jul 08 02:05:32 PDT 2014
2014-07-08 02:06:36:754 220 sec: 3500238 operations; 18757.72 current ops/sec; [UPDATE AverageLatency(us)=650061] [INSERT AverageLatency(us)=3573.64] [CLEANUP AverageLatency(us)=650078.75] 
2014-07-08 02:06:46:788 230 sec: 3781553 operations; 28036.18 current ops/sec;  [INSERT AverageLatency(us)=355.48]  
2014-07-08 02:06:56:789 240 sec: 3955750 operations; 17417.96 current ops/sec;  [INSERT AverageLatency(us)=243.92]  
2014-07-08 02:07:07:395 251 sec: 4113721 operations; 14894.49 current ops/sec;  [INSERT AverageLatency(us)=921.37]  
2014-07-08 02:07:17:396 261 sec: 4330073 operations; 21633.04 current ops/sec;  [INSERT AverageLatency(us)=467.92]  
2014-07-08 02:07:27:396 271 sec: 4599971 operations; 26989.8 current ops/sec; [UPDATE AverageLatency(us)=3398570] [INSERT AverageLatency(us)=244.84] [CLEANUP AverageLatency(us)=3398592] 
2014-07-08 02:07:37:397 281 sec: 4953284 operations; 35327.77 current ops/sec; [UPDATE AverageLatency(us)=1041363] [INSERT AverageLatency(us)=135.78] [CLEANUP AverageLatency(us)=1041383] 
2014-07-08 02:07:40:792 284 sec: 5000000 operations; 13760.24 current ops/sec; [UPDATE AverageLatency(us)=624318] [INSERT AverageLatency(us)=138.17] [CLEANUP AverageLatency(us)=624339.75] 
+ echo 'end time:'
end time:
++ date +%s
+ echo 1404810461
1404810461
+ echo 'sleeping for 10 sec ...'
sleeping for 10 sec ...
+ sleep 10
+ true
+ echo 'random read...'
random read...
+ echo 'starting time: '
starting time: 
++ date +%s
+ echo 1404810471
1404810471
++ echo /home/hadoop/git/physical_design/caseresult/case31/random_read_1.dat
+ ./random_read 5000000 20 /home/hadoop/git/physical_design/caseresult/case31/random_read_1.dat
Loading workload...
Starting test.
2014-07-08 02:07:53:683 0 sec: 0 operations; 
14/07/08 02:07:54 INFO zookeeper.ZooKeeper: Client environment:zookeeper.version=3.4.6-1569965, built on 02/20/2014 09:09 GMT
14/07/08 02:07:54 INFO zookeeper.ZooKeeper: Client environment:host.name=sceplus-vm48.almaden.ibm.com
14/07/08 02:07:54 INFO zookeeper.ZooKeeper: Client environment:java.version=1.6.0_31
14/07/08 02:07:54 INFO zookeeper.ZooKeeper: Client environment:java.vendor=Sun Microsystems Inc.
14/07/08 02:07:54 INFO zookeeper.ZooKeeper: Client environment:java.home=/usr/lib/jvm/java-6-openjdk-amd64/jre
14/07/08 02:07:54 INFO zookeeper.ZooKeeper: Client environment:java.class.path=/home/hadoop/ycsb-0.1.4/hbase-binding/lib/hbase-binding-0.1.4.jar:/home/hadoop/ycsb-0.1.4/hbase-binding/conf:/home/hadoop/ycsb-0.1.4/jdbc-binding/conf:/home/hadoop/ycsb-0.1.4/core/lib/core-0.1.4.jar
14/07/08 02:07:54 INFO zookeeper.ZooKeeper: Client environment:java.library.path=/usr/lib/jvm/java-6-openjdk-amd64/jre/lib/amd64/server:/usr/lib/jvm/java-6-openjdk-amd64/jre/lib/amd64:/usr/lib/jvm/java-6-openjdk-amd64/jre/../lib/amd64:/usr/java/packages/lib/amd64:/usr/lib/jni:/lib:/usr/lib
14/07/08 02:07:54 INFO zookeeper.ZooKeeper: Client environment:java.io.tmpdir=/tmp
14/07/08 02:07:54 INFO zookeeper.ZooKeeper: Client environment:java.compiler=<NA>
14/07/08 02:07:54 INFO zookeeper.ZooKeeper: Client environment:os.name=Linux
14/07/08 02:07:54 INFO zookeeper.ZooKeeper: Client environment:os.arch=amd64
14/07/08 02:07:54 INFO zookeeper.ZooKeeper: Client environment:os.version=3.13.0-24-generic
14/07/08 02:07:54 INFO zookeeper.ZooKeeper: Client environment:user.name=hadoop
14/07/08 02:07:54 INFO zookeeper.ZooKeeper: Client environment:user.home=/home/hadoop
14/07/08 02:07:54 INFO zookeeper.ZooKeeper: Client environment:user.dir=/home/hadoop/ycsb-0.1.4
14/07/08 02:07:54 INFO zookeeper.ZooKeeper: Initiating client connection, connectString=sceplus-vm49.almaden.ibm.com:2181,sceplus-vm48.almaden.ibm.com:2181 sessionTimeout=90000 watcher=hconnection-0x7fc8a9a0, quorum=sceplus-vm49.almaden.ibm.com:2181,sceplus-vm48.almaden.ibm.com:2181, baseZNode=/hbase
14/07/08 02:07:54 INFO zookeeper.RecoverableZooKeeper: Process identifier=hconnection-0x7fc8a9a0 connecting to ZooKeeper ensemble=sceplus-vm49.almaden.ibm.com:2181,sceplus-vm48.almaden.ibm.com:2181
14/07/08 02:07:54 INFO zookeeper.ClientCnxn: Opening socket connection to server slave1/9.1.143.59:2181. Will not attempt to authenticate using SASL (unknown error)
14/07/08 02:07:54 INFO zookeeper.ClientCnxn: Socket connection established to slave1/9.1.143.59:2181, initiating session
14/07/08 02:07:54 INFO zookeeper.ClientCnxn: Session establishment complete on server slave1/9.1.143.59:2181, sessionid = 0x1471530370b000d, negotiated timeout = 90000
2014-07-08 02:08:03:685 10 sec: 79624 operations; 7952.06 current ops/sec; [READ AverageLatency(us)=2485.85] 
2014-07-08 02:08:13:691 20 sec: 229203 operations; 14948.93 current ops/sec; [READ AverageLatency(us)=1332.77] 
2014-07-08 02:08:23:692 30 sec: 376354 operations; 14713.63 current ops/sec; [READ AverageLatency(us)=1353.93] 
2014-07-08 02:08:33:693 40 sec: 489414 operations; 11304.87 current ops/sec; [READ AverageLatency(us)=1761.55] 
2014-07-08 02:08:43:693 50 sec: 623241 operations; 13382.7 current ops/sec; [READ AverageLatency(us)=1488.72] 
2014-07-08 02:08:53:717 60 sec: 789097 operations; 16545.89 current ops/sec; [READ AverageLatency(us)=1203.07] 
2014-07-08 02:09:03:718 70 sec: 961418 operations; 17230.38 current ops/sec; [READ AverageLatency(us)=1155.03] 
2014-07-08 02:09:13:718 80 sec: 1142502 operations; 18108.4 current ops/sec; [READ AverageLatency(us)=1099.34] 
2014-07-08 02:09:23:719 90 sec: 1317426 operations; 17490.65 current ops/sec; [READ AverageLatency(us)=1138.26] 
2014-07-08 02:09:33:720 100 sec: 1489110 operations; 17168.4 current ops/sec; [READ AverageLatency(us)=1159.42] 
2014-07-08 02:09:43:720 110 sec: 1651838 operations; 16271.17 current ops/sec; [READ AverageLatency(us)=1223.37] 
2014-07-08 02:09:53:721 120 sec: 1815467 operations; 16361.26 current ops/sec; [READ AverageLatency(us)=1215.63] 
2014-07-08 02:10:03:722 130 sec: 1991958 operations; 17647.34 current ops/sec; [READ AverageLatency(us)=1128.45] 
2014-07-08 02:10:13:722 140 sec: 2173398 operations; 18144 current ops/sec; [READ AverageLatency(us)=1096.92] 
2014-07-08 02:10:23:723 150 sec: 2354440 operations; 18102.39 current ops/sec; [READ AverageLatency(us)=1099.51] 
2014-07-08 02:10:33:723 160 sec: 2536699 operations; 18225.9 current ops/sec; [READ AverageLatency(us)=1092.11] 
2014-07-08 02:10:43:724 170 sec: 2721755 operations; 18503.75 current ops/sec; [READ AverageLatency(us)=1075.66] 
2014-07-08 02:10:53:725 180 sec: 2904785 operations; 18301.17 current ops/sec; [READ AverageLatency(us)=1087.54] 
2014-07-08 02:11:03:726 190 sec: 3091066 operations; 18626.24 current ops/sec; [READ AverageLatency(us)=1068.62] 
2014-07-08 02:11:13:726 200 sec: 3274061 operations; 18299.5 current ops/sec; [READ AverageLatency(us)=1087.86] 
2014-07-08 02:11:23:727 210 sec: 3452357 operations; 17827.82 current ops/sec; [READ AverageLatency(us)=1113.91] 
2014-07-08 02:11:33:728 220 sec: 3626028 operations; 17365.36 current ops/sec; [READ AverageLatency(us)=1148.26] 
2014-07-08 02:11:43:824 230 sec: 3794081 operations; 16645.5 current ops/sec; [READ AverageLatency(us)=1190.84] 
2014-07-08 02:11:53:824 240 sec: 3970013 operations; 17593.2 current ops/sec; [READ AverageLatency(us)=1136.45] 
2014-07-08 02:12:03:825 250 sec: 4144478 operations; 17444.76 current ops/sec; [READ AverageLatency(us)=1140.77] 
2014-07-08 02:12:13:826 260 sec: 4324870 operations; 18037.4 current ops/sec; [READ AverageLatency(us)=1103.33] 
2014-07-08 02:12:23:827 270 sec: 4501229 operations; 17634.14 current ops/sec; [READ AverageLatency(us)=1127.17] 
2014-07-08 02:12:33:828 280 sec: 4681071 operations; 17982.4 current ops/sec; [READ AverageLatency(us)=1107.24] 
2014-07-08 02:12:43:829 290 sec: 4862993 operations; 18192.2 current ops/sec; [READ AverageLatency(us)=1095.45] 
2014-07-08 02:12:51:765 298 sec: 5000000 operations; 17261.81 current ops/sec; [UPDATE AverageLatency(us)=10.4] [READ AverageLatency(us)=1112.09] [CLEANUP AverageLatency(us)=197.1] 
+ echo 'end time:'
end time:
++ date +%s
+ echo 1404810772
1404810772
+ echo 'sleeping for 10 sec ...'
sleeping for 10 sec ...
+ sleep 10
+ true
+ echo workloada...
workloada...
+ echo 'start time: '
start time: 
++ date +%s
+ echo 1404810782
1404810782
++ echo /home/hadoop/git/physical_design/caseresult/case31/workloada_1.dat
+ ./workloada 5000000 20 /home/hadoop/git/physical_design/caseresult/case31/workloada_1.dat
Loading workload...
Starting test.
2014-07-08 02:13:03:169 0 sec: 0 operations; 
14/07/08 02:13:03 INFO zookeeper.ZooKeeper: Client environment:zookeeper.version=3.4.6-1569965, built on 02/20/2014 09:09 GMT
14/07/08 02:13:03 INFO zookeeper.ZooKeeper: Client environment:host.name=sceplus-vm48.almaden.ibm.com
14/07/08 02:13:03 INFO zookeeper.ZooKeeper: Client environment:java.version=1.6.0_31
14/07/08 02:13:03 INFO zookeeper.ZooKeeper: Client environment:java.vendor=Sun Microsystems Inc.
14/07/08 02:13:03 INFO zookeeper.ZooKeeper: Client environment:java.home=/usr/lib/jvm/java-6-openjdk-amd64/jre
14/07/08 02:13:03 INFO zookeeper.ZooKeeper: Client environment:java.class.path=/home/hadoop/ycsb-0.1.4/hbase-binding/lib/hbase-binding-0.1.4.jar:/home/hadoop/ycsb-0.1.4/hbase-binding/conf:/home/hadoop/ycsb-0.1.4/jdbc-binding/conf:/home/hadoop/ycsb-0.1.4/core/lib/core-0.1.4.jar
14/07/08 02:13:03 INFO zookeeper.ZooKeeper: Client environment:java.library.path=/usr/lib/jvm/java-6-openjdk-amd64/jre/lib/amd64/server:/usr/lib/jvm/java-6-openjdk-amd64/jre/lib/amd64:/usr/lib/jvm/java-6-openjdk-amd64/jre/../lib/amd64:/usr/java/packages/lib/amd64:/usr/lib/jni:/lib:/usr/lib
14/07/08 02:13:03 INFO zookeeper.ZooKeeper: Client environment:java.io.tmpdir=/tmp
14/07/08 02:13:03 INFO zookeeper.ZooKeeper: Client environment:java.compiler=<NA>
14/07/08 02:13:03 INFO zookeeper.ZooKeeper: Client environment:os.name=Linux
14/07/08 02:13:03 INFO zookeeper.ZooKeeper: Client environment:os.arch=amd64
14/07/08 02:13:03 INFO zookeeper.ZooKeeper: Client environment:os.version=3.13.0-24-generic
14/07/08 02:13:03 INFO zookeeper.ZooKeeper: Client environment:user.name=hadoop
14/07/08 02:13:03 INFO zookeeper.ZooKeeper: Client environment:user.home=/home/hadoop
14/07/08 02:13:03 INFO zookeeper.ZooKeeper: Client environment:user.dir=/home/hadoop/ycsb-0.1.4
14/07/08 02:13:03 INFO zookeeper.ZooKeeper: Initiating client connection, connectString=sceplus-vm49.almaden.ibm.com:2181,sceplus-vm48.almaden.ibm.com:2181 sessionTimeout=90000 watcher=hconnection-0xc2c915f, quorum=sceplus-vm49.almaden.ibm.com:2181,sceplus-vm48.almaden.ibm.com:2181, baseZNode=/hbase
14/07/08 02:13:03 INFO zookeeper.RecoverableZooKeeper: Process identifier=hconnection-0xc2c915f connecting to ZooKeeper ensemble=sceplus-vm49.almaden.ibm.com:2181,sceplus-vm48.almaden.ibm.com:2181
14/07/08 02:13:03 INFO zookeeper.ClientCnxn: Opening socket connection to server sceplus-vm48.almaden.ibm.com/9.1.143.58:2181. Will not attempt to authenticate using SASL (unknown error)
14/07/08 02:13:03 INFO zookeeper.ClientCnxn: Socket connection established to sceplus-vm48.almaden.ibm.com/9.1.143.58:2181, initiating session
14/07/08 02:13:03 INFO zookeeper.ClientCnxn: Session establishment complete on server sceplus-vm48.almaden.ibm.com/9.1.143.58:2181, sessionid = 0x4715303986000c, negotiated timeout = 90000
2014-07-08 02:13:13:171 10 sec: 21895 operations; 2189.06 current ops/sec; [INSERT AverageLatency(us)=118.77] [READ AverageLatency(us)=5161.35] 
2014-07-08 02:13:23:173 20 sec: 80181 operations; 5828.02 current ops/sec; [INSERT AverageLatency(us)=20.6] [READ AverageLatency(us)=2934.08] 
2014-07-08 02:13:33:173 30 sec: 165892 operations; 8570.24 current ops/sec; [INSERT AverageLatency(us)=16.66] [READ AverageLatency(us)=2501.51] 
2014-07-08 02:13:43:174 40 sec: 267201 operations; 10129.89 current ops/sec; [INSERT AverageLatency(us)=13.21] [READ AverageLatency(us)=2200.74] 
2014-07-08 02:13:53:175 50 sec: 385454 operations; 11824.12 current ops/sec; [INSERT AverageLatency(us)=13.42] [READ AverageLatency(us)=1851.31] 
2014-07-08 02:14:03:176 60 sec: 514315 operations; 12884.81 current ops/sec; [INSERT AverageLatency(us)=13.07] [READ AverageLatency(us)=1712.91] 
2014-07-08 02:14:13:176 70 sec: 666882 operations; 15256.7 current ops/sec; [INSERT AverageLatency(us)=16.6] [READ AverageLatency(us)=1438.12] 
2014-07-08 02:14:23:177 80 sec: 810216 operations; 14331.97 current ops/sec; [INSERT AverageLatency(us)=19.24] [READ AverageLatency(us)=1536.82] 
2014-07-08 02:14:33:178 90 sec: 930363 operations; 12013.5 current ops/sec; [INSERT AverageLatency(us)=36.8] [READ AverageLatency(us)=1807.32] 
2014-07-08 02:14:43:178 100 sec: 1023501 operations; 9313.8 current ops/sec; [INSERT AverageLatency(us)=33.32] [READ AverageLatency(us)=2405.91] 
2014-07-08 02:14:53:179 110 sec: 1164073 operations; 14055.79 current ops/sec; [INSERT AverageLatency(us)=20.11] [READ AverageLatency(us)=1566.63] 
2014-07-08 02:15:03:180 120 sec: 1312931 operations; 14884.31 current ops/sec; [INSERT AverageLatency(us)=23.46] [READ AverageLatency(us)=1481.41] 
2014-07-08 02:15:13:180 130 sec: 1470705 operations; 15777.4 current ops/sec; [INSERT AverageLatency(us)=17.79] [READ AverageLatency(us)=1398.23] 
2014-07-08 02:15:23:181 140 sec: 1623086 operations; 15236.58 current ops/sec; [INSERT AverageLatency(us)=18.49] [READ AverageLatency(us)=1448.82] 
2014-07-08 02:15:33:182 150 sec: 1782658 operations; 15955.6 current ops/sec; [INSERT AverageLatency(us)=16.88] [READ AverageLatency(us)=1383.06] 
2014-07-08 02:15:43:182 160 sec: 1876179 operations; 9352.1 current ops/sec; [INSERT AverageLatency(us)=34.89] [READ AverageLatency(us)=2364.74] 
2014-07-08 02:15:53:183 170 sec: 1987727 operations; 11153.68 current ops/sec; [INSERT AverageLatency(us)=23.52] [READ AverageLatency(us)=1979.9] 
2014-07-08 02:16:03:184 180 sec: 2147595 operations; 15985.2 current ops/sec; [INSERT AverageLatency(us)=16.71] [READ AverageLatency(us)=1379.22] 
2014-07-08 02:16:13:185 190 sec: 2310554 operations; 16294.27 current ops/sec; [INSERT AverageLatency(us)=19.8] [READ AverageLatency(us)=1353.26] 
2014-07-08 02:16:23:186 200 sec: 2480379 operations; 16980.8 current ops/sec; [INSERT AverageLatency(us)=16.59] [READ AverageLatency(us)=1289.62] 
2014-07-08 02:16:33:187 210 sec: 2649930 operations; 16953.4 current ops/sec; [INSERT AverageLatency(us)=19.27] [READ AverageLatency(us)=1310.08] 
2014-07-08 02:16:43:187 220 sec: 2801412 operations; 15148.2 current ops/sec; [INSERT AverageLatency(us)=26.38] [READ AverageLatency(us)=1454.8] 
2014-07-08 02:16:53:188 230 sec: 2932132 operations; 13070.69 current ops/sec; [INSERT AverageLatency(us)=24.71] [READ AverageLatency(us)=1687.9] 
2014-07-08 02:17:03:189 240 sec: 3112391 operations; 18024.1 current ops/sec; [INSERT AverageLatency(us)=22.67] [READ AverageLatency(us)=1223.2] 
2014-07-08 02:17:13:190 250 sec: 3284751 operations; 17234.28 current ops/sec; [INSERT AverageLatency(us)=16.99] [READ AverageLatency(us)=1279.59] 
2014-07-08 02:17:23:191 260 sec: 3472747 operations; 18797.72 current ops/sec; [INSERT AverageLatency(us)=13.4] [READ AverageLatency(us)=1173.33] 
2014-07-08 02:17:33:192 270 sec: 3656461 operations; 18369.56 current ops/sec; [INSERT AverageLatency(us)=15.79] [READ AverageLatency(us)=1201.63] 
2014-07-08 02:17:43:193 280 sec: 3776629 operations; 12015.6 current ops/sec; [INSERT AverageLatency(us)=20.74] [READ AverageLatency(us)=1834.05] 
2014-07-08 02:17:53:194 290 sec: 3927232 operations; 15058.79 current ops/sec; [INSERT AverageLatency(us)=17.33] [READ AverageLatency(us)=1465.54] 
2014-07-08 02:18:03:195 300 sec: 4109653 operations; 18240.28 current ops/sec; [INSERT AverageLatency(us)=15.26] [READ AverageLatency(us)=1208.83] 
2014-07-08 02:18:13:195 310 sec: 4295055 operations; 18540.2 current ops/sec; [INSERT AverageLatency(us)=13.94] [READ AverageLatency(us)=1191.35] 
2014-07-08 02:18:23:196 320 sec: 4477769 operations; 18269.57 current ops/sec; [INSERT AverageLatency(us)=14.12] [READ AverageLatency(us)=1205.02] 
2014-07-08 02:18:33:197 330 sec: 4659279 operations; 18149.19 current ops/sec; [INSERT AverageLatency(us)=12.87] [READ AverageLatency(us)=1215.38] 
2014-07-08 02:18:43:198 340 sec: 4770900 operations; 11160.98 current ops/sec; [INSERT AverageLatency(us)=20.79] [READ AverageLatency(us)=1979.59] 
2014-07-08 02:18:53:199 350 sec: 4933822 operations; 16290.57 current ops/sec; [INSERT AverageLatency(us)=13.78] [READ AverageLatency(us)=1357.24] 
2014-07-08 02:18:58:936 355 sec: 5000000 operations; 11535.3 current ops/sec; [UPDATE AverageLatency(us)=149296.9] [INSERT AverageLatency(us)=15.62] [READ AverageLatency(us)=1448.65] [CLEANUP AverageLatency(us)=149463.3] 
+ echo 'end time:'
end time:
++ date +%s
+ echo 1404811139
1404811139
+ echo 'sleeping for 10 sec ...'
sleeping for 10 sec ...
+ sleep 10
+ true
+ echo workloadb...
workloadb...
+ echo 'start time: '
start time: 
++ date +%s
+ echo 1404811149
1404811149
++ echo /home/hadoop/git/physical_design/caseresult/case31/workloadb_1.dat
+ ./workloadb 5000000 20 /home/hadoop/git/physical_design/caseresult/case31/workloadb_1.dat
Loading workload...
Starting test.
2014-07-08 02:19:09:977 0 sec: 0 operations; 
14/07/08 02:19:10 INFO zookeeper.ZooKeeper: Client environment:zookeeper.version=3.4.6-1569965, built on 02/20/2014 09:09 GMT
14/07/08 02:19:10 INFO zookeeper.ZooKeeper: Client environment:host.name=sceplus-vm48.almaden.ibm.com
14/07/08 02:19:10 INFO zookeeper.ZooKeeper: Client environment:java.version=1.6.0_31
14/07/08 02:19:10 INFO zookeeper.ZooKeeper: Client environment:java.vendor=Sun Microsystems Inc.
14/07/08 02:19:10 INFO zookeeper.ZooKeeper: Client environment:java.home=/usr/lib/jvm/java-6-openjdk-amd64/jre
14/07/08 02:19:10 INFO zookeeper.ZooKeeper: Client environment:java.class.path=/home/hadoop/ycsb-0.1.4/hbase-binding/lib/hbase-binding-0.1.4.jar:/home/hadoop/ycsb-0.1.4/hbase-binding/conf:/home/hadoop/ycsb-0.1.4/jdbc-binding/conf:/home/hadoop/ycsb-0.1.4/core/lib/core-0.1.4.jar
14/07/08 02:19:10 INFO zookeeper.ZooKeeper: Client environment:java.library.path=/usr/lib/jvm/java-6-openjdk-amd64/jre/lib/amd64/server:/usr/lib/jvm/java-6-openjdk-amd64/jre/lib/amd64:/usr/lib/jvm/java-6-openjdk-amd64/jre/../lib/amd64:/usr/java/packages/lib/amd64:/usr/lib/jni:/lib:/usr/lib
14/07/08 02:19:10 INFO zookeeper.ZooKeeper: Client environment:java.io.tmpdir=/tmp
14/07/08 02:19:10 INFO zookeeper.ZooKeeper: Client environment:java.compiler=<NA>
14/07/08 02:19:10 INFO zookeeper.ZooKeeper: Client environment:os.name=Linux
14/07/08 02:19:10 INFO zookeeper.ZooKeeper: Client environment:os.arch=amd64
14/07/08 02:19:10 INFO zookeeper.ZooKeeper: Client environment:os.version=3.13.0-24-generic
14/07/08 02:19:10 INFO zookeeper.ZooKeeper: Client environment:user.name=hadoop
14/07/08 02:19:10 INFO zookeeper.ZooKeeper: Client environment:user.home=/home/hadoop
14/07/08 02:19:10 INFO zookeeper.ZooKeeper: Client environment:user.dir=/home/hadoop/ycsb-0.1.4
14/07/08 02:19:10 INFO zookeeper.ZooKeeper: Initiating client connection, connectString=sceplus-vm49.almaden.ibm.com:2181,sceplus-vm48.almaden.ibm.com:2181 sessionTimeout=90000 watcher=hconnection-0x30ea3e3c, quorum=sceplus-vm49.almaden.ibm.com:2181,sceplus-vm48.almaden.ibm.com:2181, baseZNode=/hbase
14/07/08 02:19:10 INFO zookeeper.RecoverableZooKeeper: Process identifier=hconnection-0x30ea3e3c connecting to ZooKeeper ensemble=sceplus-vm49.almaden.ibm.com:2181,sceplus-vm48.almaden.ibm.com:2181
14/07/08 02:19:10 INFO zookeeper.ClientCnxn: Opening socket connection to server slave1/9.1.143.59:2181. Will not attempt to authenticate using SASL (unknown error)
14/07/08 02:19:10 INFO zookeeper.ClientCnxn: Socket connection established to slave1/9.1.143.59:2181, initiating session
14/07/08 02:19:10 INFO zookeeper.ClientCnxn: Session establishment complete on server slave1/9.1.143.59:2181, sessionid = 0x1471530370b000e, negotiated timeout = 90000
2014-07-08 02:19:19:981 10 sec: 19023 operations; 1901.35 current ops/sec; [INSERT AverageLatency(us)=1684.16] [READ AverageLatency(us)=5571.34] 
2014-07-08 02:19:29:987 20 sec: 189637 operations; 17051.17 current ops/sec; [INSERT AverageLatency(us)=21.27] [READ AverageLatency(us)=1575.92] 
2014-07-08 02:19:39:988 30 sec: 336023 operations; 14637.14 current ops/sec; [INSERT AverageLatency(us)=38.73] [READ AverageLatency(us)=2642.88] 
2014-07-08 02:19:49:989 40 sec: 493880 operations; 15784.12 current ops/sec; [INSERT AverageLatency(us)=17] [READ AverageLatency(us)=2480.61] 
2014-07-08 02:19:59:993 50 sec: 671339 operations; 17738.8 current ops/sec; [INSERT AverageLatency(us)=24.94] [READ AverageLatency(us)=2207.44] 
2014-07-08 02:20:09:994 60 sec: 880345 operations; 20898.51 current ops/sec; [INSERT AverageLatency(us)=19.02] [READ AverageLatency(us)=1878.29] 
2014-07-08 02:20:19:995 70 sec: 1044657 operations; 16429.56 current ops/sec; [INSERT AverageLatency(us)=18.13] [READ AverageLatency(us)=2401.97] 
2014-07-08 02:20:29:996 80 sec: 1195090 operations; 15041.8 current ops/sec; [INSERT AverageLatency(us)=87.17] [READ AverageLatency(us)=2561.26] 
2014-07-08 02:20:39:997 90 sec: 1397463 operations; 20235.28 current ops/sec; [INSERT AverageLatency(us)=21.96] [READ AverageLatency(us)=1939.6] 
2014-07-08 02:20:49:998 100 sec: 1588617 operations; 19115.4 current ops/sec; [INSERT AverageLatency(us)=16.53] [READ AverageLatency(us)=2061.51] 
2014-07-08 02:20:59:998 110 sec: 1799572 operations; 21093.39 current ops/sec; [INSERT AverageLatency(us)=16.86] [READ AverageLatency(us)=1868.37] 
2014-07-08 02:21:09:999 120 sec: 1948907 operations; 14932.01 current ops/sec; [INSERT AverageLatency(us)=14.71] [READ AverageLatency(us)=2645.56] 
2014-07-08 02:21:19:999 130 sec: 2091801 operations; 14289.4 current ops/sec; [INSERT AverageLatency(us)=17.8] [READ AverageLatency(us)=2775.54] 
2014-07-08 02:21:30:000 140 sec: 2281647 operations; 18982.7 current ops/sec; [INSERT AverageLatency(us)=14.61] [READ AverageLatency(us)=2077.94] 
2014-07-08 02:21:40:001 150 sec: 2453648 operations; 17198.38 current ops/sec; [INSERT AverageLatency(us)=12.49] [READ AverageLatency(us)=2296] 
2014-07-08 02:21:50:001 160 sec: 2621802 operations; 16815.4 current ops/sec; [INSERT AverageLatency(us)=17.58] [READ AverageLatency(us)=2349.17] 
2014-07-08 02:22:00:002 170 sec: 2785923 operations; 16410.46 current ops/sec; [INSERT AverageLatency(us)=16.92] [READ AverageLatency(us)=2412.9] 
2014-07-08 02:22:10:003 180 sec: 2796997 operations; 1107.4 current ops/sec; [INSERT AverageLatency(us)=13.64] [READ AverageLatency(us)=3285.03] 
2014-07-08 02:22:20:003 190 sec: 2796997 operations; 0 current ops/sec;   
2014-07-08 02:22:30:004 200 sec: 2797878 operations; 88.09 current ops/sec; [INSERT AverageLatency(us)=15.79] [READ AverageLatency(us)=1273620.44] 
2014-07-08 02:22:40:130 210 sec: 2889332 operations; 9031.6 current ops/sec; [INSERT AverageLatency(us)=21.04] [READ AverageLatency(us)=4388.05] 
2014-07-08 02:22:50:131 220 sec: 3008475 operations; 11913.11 current ops/sec; [INSERT AverageLatency(us)=20.02] [READ AverageLatency(us)=3362.07] 
2014-07-08 02:23:00:131 230 sec: 3129464 operations; 12098.9 current ops/sec; [INSERT AverageLatency(us)=21.42] [READ AverageLatency(us)=3273.2] 
2014-07-08 02:23:10:132 240 sec: 3226227 operations; 9675.33 current ops/sec; [INSERT AverageLatency(us)=25.42] [READ AverageLatency(us)=4084.82] 
2014-07-08 02:23:20:143 250 sec: 3372150 operations; 14576.27 current ops/sec; [INSERT AverageLatency(us)=14.82] [READ AverageLatency(us)=2711.63] 
2014-07-08 02:23:30:148 260 sec: 3450084 operations; 7790.28 current ops/sec; [INSERT AverageLatency(us)=36.73] [READ AverageLatency(us)=5068.98] 
2014-07-08 02:23:40:148 270 sec: 3553113 operations; 10301.87 current ops/sec; [INSERT AverageLatency(us)=14.75] [READ AverageLatency(us)=3865.56] 
2014-07-08 02:23:50:154 280 sec: 3623836 operations; 7068.06 current ops/sec; [INSERT AverageLatency(us)=18.85] [READ AverageLatency(us)=5556.39] 
2014-07-08 02:24:00:155 290 sec: 3728951 operations; 10510.45 current ops/sec; [INSERT AverageLatency(us)=16.97] [READ AverageLatency(us)=3772.43] 
2014-07-08 02:24:10:172 300 sec: 3800183 operations; 7111.11 current ops/sec; [INSERT AverageLatency(us)=27.17] [READ AverageLatency(us)=5533.45] 
2014-07-08 02:24:20:188 310 sec: 3900157 operations; 9981.43 current ops/sec; [INSERT AverageLatency(us)=16.88] [READ AverageLatency(us)=3995.21] 
2014-07-08 02:24:30:189 320 sec: 3958402 operations; 5823.92 current ops/sec; [INSERT AverageLatency(us)=37.24] [READ AverageLatency(us)=6037.79] 
2014-07-08 02:24:40:198 330 sec: 3977765 operations; 1934.56 current ops/sec; [INSERT AverageLatency(us)=30.2] [READ AverageLatency(us)=22814.98] 
2014-07-08 02:24:50:199 340 sec: 4048746 operations; 7097.39 current ops/sec; [INSERT AverageLatency(us)=27.92] [READ AverageLatency(us)=5621.92] 
2014-07-08 02:25:00:200 350 sec: 4105401 operations; 5664.93 current ops/sec; [INSERT AverageLatency(us)=59.1] [READ AverageLatency(us)=6959.59] 
2014-07-08 02:25:10:217 360 sec: 4159903 operations; 5440.95 current ops/sec; [INSERT AverageLatency(us)=37.03] [READ AverageLatency(us)=7374.98] 
2014-07-08 02:25:20:228 370 sec: 4236401 operations; 7641.39 current ops/sec; [INSERT AverageLatency(us)=33.18] [READ AverageLatency(us)=5211.09] 
2014-07-08 02:25:30:229 380 sec: 4322586 operations; 8617.64 current ops/sec; [INSERT AverageLatency(us)=24.72] [READ AverageLatency(us)=4595.18] 
2014-07-08 02:25:40:242 390 sec: 4391325 operations; 6864.98 current ops/sec; [INSERT AverageLatency(us)=18.82] [READ AverageLatency(us)=5780.5] 
2014-07-08 02:25:50:245 400 sec: 4466109 operations; 7476.16 current ops/sec; [INSERT AverageLatency(us)=26.95] [READ AverageLatency(us)=5301.34] 
2014-07-08 02:26:00:578 410 sec: 4541975 operations; 7342.11 current ops/sec; [INSERT AverageLatency(us)=20.68] [READ AverageLatency(us)=5052.72] 
2014-07-08 02:26:11:199 421 sec: 4610034 operations; 6407.97 current ops/sec; [INSERT AverageLatency(us)=14.85] [READ AverageLatency(us)=5404.78] 
2014-07-08 02:26:21:200 431 sec: 4681524 operations; 7148.29 current ops/sec; [INSERT AverageLatency(us)=16.94] [READ AverageLatency(us)=6637.19] 
2014-07-08 02:26:31:250 441 sec: 4737338 operations; 5553.63 current ops/sec; [INSERT AverageLatency(us)=55.19] [READ AverageLatency(us)=7195.02] 
2014-07-08 02:26:41:264 451 sec: 4839466 operations; 10198.52 current ops/sec; [INSERT AverageLatency(us)=15.7] [READ AverageLatency(us)=3897.64] 
2014-07-08 02:26:51:282 461 sec: 4901192 operations; 6161.51 current ops/sec; [INSERT AverageLatency(us)=26.04] [READ AverageLatency(us)=6417.96] 
2014-07-08 02:27:01:349 471 sec: 4960156 operations; 5857.74 current ops/sec; [UPDATE AverageLatency(us)=294213] [INSERT AverageLatency(us)=26.01] [READ AverageLatency(us)=6797.95] [CLEANUP AverageLatency(us)=307377] 
2014-07-08 02:27:11:361 481 sec: 4996510 operations; 3630.68 current ops/sec; [UPDATE AverageLatency(us)=303340.09] [INSERT AverageLatency(us)=17.22] [READ AverageLatency(us)=6887.3] [CLEANUP AverageLatency(us)=304850.73] 
2014-07-08 02:27:13:894 483 sec: 5000000 operations; 1377.81 current ops/sec; [UPDATE AverageLatency(us)=214639.62] [INSERT AverageLatency(us)=11.55] [READ AverageLatency(us)=4428.47] [CLEANUP AverageLatency(us)=214656.38] 
+ echo 'end time:'
end time:
++ date +%s
+ echo 1404811635
1404811635
+ echo 'sleeping for 10 sec ...'
sleeping for 10 sec ...
+ sleep 10
+ true
+ echo workloadc...
workloadc...
+ echo 'start time: '
start time: 
++ date +%s
+ echo 1404811645
1404811645
++ echo /home/hadoop/git/physical_design/caseresult/case31/workloadc_1.dat
+ ./workloadc 5000000 20 /home/hadoop/git/physical_design/caseresult/case31/workloadc_1.dat
Loading workload...
Starting test.
2014-07-08 02:27:26:631 0 sec: 0 operations; 
14/07/08 02:27:27 INFO zookeeper.ZooKeeper: Client environment:zookeeper.version=3.4.6-1569965, built on 02/20/2014 09:09 GMT
14/07/08 02:27:27 INFO zookeeper.ZooKeeper: Client environment:host.name=sceplus-vm48.almaden.ibm.com
14/07/08 02:27:27 INFO zookeeper.ZooKeeper: Client environment:java.version=1.6.0_31
14/07/08 02:27:27 INFO zookeeper.ZooKeeper: Client environment:java.vendor=Sun Microsystems Inc.
14/07/08 02:27:27 INFO zookeeper.ZooKeeper: Client environment:java.home=/usr/lib/jvm/java-6-openjdk-amd64/jre
14/07/08 02:27:27 INFO zookeeper.ZooKeeper: Client environment:java.class.path=/home/hadoop/ycsb-0.1.4/hbase-binding/lib/hbase-binding-0.1.4.jar:/home/hadoop/ycsb-0.1.4/hbase-binding/conf:/home/hadoop/ycsb-0.1.4/jdbc-binding/conf:/home/hadoop/ycsb-0.1.4/core/lib/core-0.1.4.jar
14/07/08 02:27:27 INFO zookeeper.ZooKeeper: Client environment:java.library.path=/usr/lib/jvm/java-6-openjdk-amd64/jre/lib/amd64/server:/usr/lib/jvm/java-6-openjdk-amd64/jre/lib/amd64:/usr/lib/jvm/java-6-openjdk-amd64/jre/../lib/amd64:/usr/java/packages/lib/amd64:/usr/lib/jni:/lib:/usr/lib
14/07/08 02:27:27 INFO zookeeper.ZooKeeper: Client environment:java.io.tmpdir=/tmp
14/07/08 02:27:27 INFO zookeeper.ZooKeeper: Client environment:java.compiler=<NA>
14/07/08 02:27:27 INFO zookeeper.ZooKeeper: Client environment:os.name=Linux
14/07/08 02:27:27 INFO zookeeper.ZooKeeper: Client environment:os.arch=amd64
14/07/08 02:27:27 INFO zookeeper.ZooKeeper: Client environment:os.version=3.13.0-24-generic
14/07/08 02:27:27 INFO zookeeper.ZooKeeper: Client environment:user.name=hadoop
14/07/08 02:27:27 INFO zookeeper.ZooKeeper: Client environment:user.home=/home/hadoop
14/07/08 02:27:27 INFO zookeeper.ZooKeeper: Client environment:user.dir=/home/hadoop/ycsb-0.1.4
14/07/08 02:27:27 INFO zookeeper.ZooKeeper: Initiating client connection, connectString=sceplus-vm49.almaden.ibm.com:2181,sceplus-vm48.almaden.ibm.com:2181 sessionTimeout=90000 watcher=hconnection-0x578ef2b6, quorum=sceplus-vm49.almaden.ibm.com:2181,sceplus-vm48.almaden.ibm.com:2181, baseZNode=/hbase
14/07/08 02:27:27 INFO zookeeper.RecoverableZooKeeper: Process identifier=hconnection-0x578ef2b6 connecting to ZooKeeper ensemble=sceplus-vm49.almaden.ibm.com:2181,sceplus-vm48.almaden.ibm.com:2181
14/07/08 02:27:27 INFO zookeeper.ClientCnxn: Opening socket connection to server slave1/9.1.143.59:2181. Will not attempt to authenticate using SASL (unknown error)
14/07/08 02:27:27 INFO zookeeper.ClientCnxn: Socket connection established to slave1/9.1.143.59:2181, initiating session
14/07/08 02:27:27 INFO zookeeper.ClientCnxn: Session establishment complete on server slave1/9.1.143.59:2181, sessionid = 0x1471530370b000f, negotiated timeout = 90000
2014-07-08 02:27:36:634 10 sec: 111226 operations; 11102.62 current ops/sec; [INSERT AverageLatency(us)=212.37] [READ AverageLatency(us)=10753.89] 
2014-07-08 02:27:46:635 20 sec: 225019 operations; 11377.02 current ops/sec; [INSERT AverageLatency(us)=29.88] [READ AverageLatency(us)=17601.76] 
2014-07-08 02:27:56:644 30 sec: 411283 operations; 18609.65 current ops/sec; [INSERT AverageLatency(us)=21.34] [READ AverageLatency(us)=11177.57] 
2014-07-08 02:28:06:645 40 sec: 547635 operations; 13635.2 current ops/sec; [INSERT AverageLatency(us)=18.79] [READ AverageLatency(us)=14450.27] 
2014-07-08 02:28:16:645 50 sec: 716959 operations; 16930.71 current ops/sec; [INSERT AverageLatency(us)=17.74] [READ AverageLatency(us)=12136.8] 
2014-07-08 02:28:26:646 60 sec: 810611 operations; 9364.26 current ops/sec; [INSERT AverageLatency(us)=16.64] [READ AverageLatency(us)=21682.76] 
2014-07-08 02:28:36:647 70 sec: 946807 operations; 13618.24 current ops/sec; [INSERT AverageLatency(us)=15.68] [READ AverageLatency(us)=14216.58] 
2014-07-08 02:28:46:648 80 sec: 1127193 operations; 18036.8 current ops/sec; [INSERT AverageLatency(us)=18.11] [READ AverageLatency(us)=10858.66] 
2014-07-08 02:28:56:649 90 sec: 1293066 operations; 16585.64 current ops/sec; [INSERT AverageLatency(us)=18.52] [READ AverageLatency(us)=11706.59] 
2014-07-08 02:29:06:671 100 sec: 1322381 operations; 2925.06 current ops/sec; [INSERT AverageLatency(us)=26.37] [READ AverageLatency(us)=13591.29] 
2014-07-08 02:29:16:680 110 sec: 1433893 operations; 11141.17 current ops/sec; [INSERT AverageLatency(us)=19.45] [READ AverageLatency(us)=32177.72] 
2014-07-08 02:29:26:680 120 sec: 1562214 operations; 12832.1 current ops/sec; [INSERT AverageLatency(us)=14.85] [READ AverageLatency(us)=15208.72] 
2014-07-08 02:29:36:681 130 sec: 1718828 operations; 15659.83 current ops/sec; [INSERT AverageLatency(us)=21.02] [READ AverageLatency(us)=12506.88] 
2014-07-08 02:29:46:682 140 sec: 1815609 operations; 9677.13 current ops/sec; [INSERT AverageLatency(us)=25.12] [READ AverageLatency(us)=20946.53] 
2014-07-08 02:29:56:683 150 sec: 1943670 operations; 12804.82 current ops/sec; [INSERT AverageLatency(us)=15.47] [READ AverageLatency(us)=15628.43] 
2014-07-08 02:30:06:706 160 sec: 2069952 operations; 12599.22 current ops/sec; [INSERT AverageLatency(us)=17.65] [READ AverageLatency(us)=15149.55] 
2014-07-08 02:30:16:707 170 sec: 2193900 operations; 12393.56 current ops/sec; [INSERT AverageLatency(us)=15.45] [READ AverageLatency(us)=16593.23] 
2014-07-08 02:30:26:730 180 sec: 2316930 operations; 12274.77 current ops/sec; [INSERT AverageLatency(us)=17.42] [READ AverageLatency(us)=16088.23] 
2014-07-08 02:30:36:732 190 sec: 2424127 operations; 10717.56 current ops/sec; [INSERT AverageLatency(us)=15.61] [READ AverageLatency(us)=18352.73] 
2014-07-08 02:30:46:733 200 sec: 2560879 operations; 13673.83 current ops/sec; [INSERT AverageLatency(us)=15.65] [READ AverageLatency(us)=14588.1] 
2014-07-08 02:30:56:743 210 sec: 2659997 operations; 9901.9 current ops/sec; [INSERT AverageLatency(us)=14.88] [READ AverageLatency(us)=19865.05] 
2014-07-08 02:31:06:755 220 sec: 2762323 operations; 10220.34 current ops/sec; [INSERT AverageLatency(us)=12.67] [READ AverageLatency(us)=16747.52] 
2014-07-08 02:31:16:757 230 sec: 2877721 operations; 11537.49 current ops/sec; [INSERT AverageLatency(us)=16.48] [READ AverageLatency(us)=17409.1] 
2014-07-08 02:31:26:804 240 sec: 2985512 operations; 10728.68 current ops/sec; [INSERT AverageLatency(us)=12.83] [READ AverageLatency(us)=21167.77] 
2014-07-08 02:31:36:819 250 sec: 3093504 operations; 10783.03 current ops/sec; [INSERT AverageLatency(us)=13.87] [READ AverageLatency(us)=17817.46] 
2014-07-08 02:31:46:821 260 sec: 3239639 operations; 14610.58 current ops/sec; [INSERT AverageLatency(us)=15.26] [READ AverageLatency(us)=13739.51] 
2014-07-08 02:31:56:836 270 sec: 3383554 operations; 14369.95 current ops/sec; [INSERT AverageLatency(us)=15.05] [READ AverageLatency(us)=13678.14] 
2014-07-08 02:32:06:840 280 sec: 3505709 operations; 12210.62 current ops/sec; [INSERT AverageLatency(us)=15.07] [READ AverageLatency(us)=15836.37] 
2014-07-08 02:32:16:848 290 sec: 3634196 operations; 12838.43 current ops/sec; [INSERT AverageLatency(us)=14.52] [READ AverageLatency(us)=15411.13] 
2014-07-08 02:32:26:851 300 sec: 3798012 operations; 16378.32 current ops/sec; [INSERT AverageLatency(us)=17.44] [READ AverageLatency(us)=11962.06] 
2014-07-08 02:32:36:851 310 sec: 3956766 operations; 15873.81 current ops/sec; [INSERT AverageLatency(us)=15.49] [READ AverageLatency(us)=12468.28] 
2014-07-08 02:32:46:861 320 sec: 4116202 operations; 15927.67 current ops/sec; [INSERT AverageLatency(us)=15.1] [READ AverageLatency(us)=12602.68] 
2014-07-08 02:32:56:864 330 sec: 4243097 operations; 12685.69 current ops/sec; [INSERT AverageLatency(us)=17.02] [READ AverageLatency(us)=15735.54] 
2014-07-08 02:33:06:866 340 sec: 4354925 operations; 11180.56 current ops/sec; [INSERT AverageLatency(us)=13.74] [READ AverageLatency(us)=17436.24] 
2014-07-08 02:33:16:873 350 sec: 4481141 operations; 12612.77 current ops/sec; [INSERT AverageLatency(us)=14.22] [READ AverageLatency(us)=15631.1] 
2014-07-08 02:33:26:879 360 sec: 4616581 operations; 13535.88 current ops/sec; [INSERT AverageLatency(us)=14.57] [READ AverageLatency(us)=14445.49] 
2014-07-08 02:33:36:882 370 sec: 4740995 operations; 12437.67 current ops/sec; [INSERT AverageLatency(us)=15.63] [READ AverageLatency(us)=15423.69] 
2014-07-08 02:33:46:882 380 sec: 4876350 operations; 13535.5 current ops/sec; [INSERT AverageLatency(us)=14.13] [READ AverageLatency(us)=14785.71] 
2014-07-08 02:33:57:029 390 sec: 4968830 operations; 9114.02 current ops/sec; [UPDATE AverageLatency(us)=480043.88] [INSERT AverageLatency(us)=13.25] [READ AverageLatency(us)=18899.23] [CLEANUP AverageLatency(us)=480817] 
2014-07-08 02:34:06:431 399 sec: 5000000 operations; 3315.25 current ops/sec; [UPDATE AverageLatency(us)=415634] [INSERT AverageLatency(us)=12.15] [READ AverageLatency(us)=10945.55] [CLEANUP AverageLatency(us)=415651] 
+ echo 'end time:'
end time:
++ date +%s
+ echo 1404812047
1404812047
+ echo 'sleeping for 10 sec ...'
sleeping for 10 sec ...
+ sleep 10
+ let counter=1+1
+ read line
+ set +x
